diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 5979283..377e581 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -7,7 +7,6 @@
 
 project
     : requirements
-      <include>$(BOOST_ROOT)
       <cxxstd>11
     ;
 
@@ -39,6 +38,7 @@ exe insertion_emplacement
 
 exe perf
     : perf.cpp
+      /boost/ptr_container//boost_ptr_container
     :
     : release
     ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index ee50c7d..127326d 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,8 +5,11 @@
 #
 # See http://www.boost.org/libs/poly_collection for library home page.
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : requirements
