diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 5256fa7..5e49184 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -6,7 +6,7 @@
 
 import mpi ;
 
-project boost/graph_parallel
+project
     : requirements <include>../src
     : source-location ../src
     ;
@@ -15,10 +15,10 @@ local optional_sources ;
 local optional_reqs ;
 
 if [ mpi.configured ]
-{    
+{
   lib boost_graph_parallel
     : mpi_process_group.cpp tag_allocator.cpp
-    : <library>../../mpi/build//boost_mpi
+    : <library>/boost/mpi//boost_mpi
       <library>/mpi//mpi [ mpi.extra-requirements ]
       <define>BOOST_GRAPH_NO_LIB=1
       <link>shared:<define>BOOST_GRAPH_DYN_LINK=1
@@ -33,11 +33,13 @@ if [ mpi.configured ]
 }
 else
 {
-  message boost_graph_parallel
+  alias boost_graph_parallel ;
+  if ! ( --without-graph_parallel in  [ modules.peek : ARGV ] )
+  {
+    message boost_graph_parallel
       : "warning: Graph library does not contain MPI-based parallel components."
       : "note: to enable them, add \"using mpi ;\" to your user-config.jam."
       : "note: to suppress this message, pass \"--without-graph_parallel\" to bjam."
       ;
+  }
 }
-
-boost-install boost_graph_parallel ;
diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index e887775..012c649 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -5,9 +5,10 @@
 #          http://www.boost.org/LICENSE_1_0.txt)
 
 
-project : requirements <library>../build//boost_graph_parallel 
-    <library>../../system/build//boost_system
-    <library>../../mpi/build//boost_mpi
+project : requirements <library>/boost/graph_parallel//boost_graph_parallel 
+    <library>/boost/system//boost_system
+    <library>/boost/mpi//boost_mpi
+    <library>/python//python
     ;
 
 exe breadth_first_search : breadth_first_search.cpp ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index b3e062e..d5671e8 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -6,11 +6,13 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt.)
 
 
-# use-project /boost/mpi : ../build ;
-
 project /boost/graph_parallel/test 
-    : requirements <library>../build//boost_graph_parallel <library>../../system/build//boost_system ;
-    
+    : requirements
+        <library>/boost/graph_parallel//boost_graph_parallel
+        <library>/boost/system//boost_system
+        <library>/python//python
+    ;
+
 import mpi : mpi-test ;
 
 if [ mpi.configured ]
@@ -46,4 +48,3 @@ test-suite graph_parallel
 }
 
 build-project ../example ;
-
