diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 7e1ae58..3fef24c 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -28,5 +28,5 @@ test-suite "circular_buffer"
     [ run space_optimized_test.cpp : : : <threading>single <define>"BOOST_CB_ENABLE_DEBUG=1" : space_optimized_test_dbg ]
     [ run soft_iterator_invalidation.cpp : : : <threading>single : ]
     [ run constant_erase_test.cpp : : : <threading>single : ]
-    [ compile bounded_buffer_comparison.cpp : <threading>multi : ]
+    [ compile bounded_buffer_comparison.cpp : <threading>multi <library>/boost/thread//boost_thread <library>/boost/timer//boost_timer : ]
   ;
