diff --git a/build/Jamfile b/build/Jamfile
index 09802d5..e77bb19 100644
--- a/build/Jamfile
+++ b/build/Jamfile
@@ -3,9 +3,12 @@
 # Distributed under the Boost Software License, Version 1.0.
 # https://www.boost.org/LICENSE_1_0.txt
 
-import ../../config/checks/config : requires ;
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
 
-project boost/charconv ;
+import config : requires ;
+
+project ;
 
 local SOURCES = from_chars.cpp to_chars.cpp ;
 
@@ -29,5 +32,3 @@ lib boost_charconv
   # usage-requirements
   : <link>shared:<define>BOOST_CHARCONV_DYN_LINK=1
 ;
-
-boost-install boost_charconv ;
diff --git a/test/Jamfile b/test/Jamfile
index c262e7a..f438274 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -3,8 +3,11 @@
 # Distributed under the Boost Software License, Version 1.0.
 # https://www.boost.org/LICENSE_1_0.txt
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project : requirements
 
@@ -57,7 +60,7 @@ run P2497.cpp ;
 run github_issue_110.cpp ;
 run github_issue_122.cpp ;
 run from_chars_string_view.cpp ;
-run github_issue_152.cpp ;
+run github_issue_152.cpp /boost/random//boost_random ;
 run github_issue_152_float128.cpp : : : [ check-target-builds ../config//has_float128 "GCC libquadmath and __float128 support" : <library>"quadmath" ] ;
 run github_issue_154.cpp ;
 #run github_issue_156.cpp ;
