diff --git a/test/Jamfile b/test/Jamfile
index 6b723cf..16c3029 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -11,7 +11,7 @@
 import testing ;
 import os ;
 
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 if [ os.environ CI ]
 {
@@ -29,11 +29,11 @@ project
         $(CI_DEFINES)
     ;
 
-local multi-threading = <library>/boost/thread
+local multi-threading = <library>/boost/thread//boost_thread
                         <threading>multi <define>BOOST_ALL_NO_LIB=1 ;
 
 test-suite phoenix_core :
-    [ run core/custom_terminal.cpp ]
+    [ run core/custom_terminal.cpp /boost/log//boost_log ]
     [ run core/intel_test.cpp ]
     [ run core/primitives_tests.cpp ]
     ;
@@ -176,10 +176,10 @@ test-suite phoenix_algorithm :
     [ run algorithm/transformation2.cpp ]
     [ run algorithm/transformation3.cpp ]
     [ run algorithm/transformation4.cpp ]
-    [ run algorithm/querying.cpp ]
-    [ run algorithm/querying_find.cpp ]
+    [ run algorithm/querying.cpp /boost/assign//boost_assign ]
+    [ run algorithm/querying_find.cpp /boost/assign//boost_assign ]
     [ run algorithm/querying2.cpp ]
-    [ run algorithm/querying_find2.cpp ]
+    [ run algorithm/querying_find2.cpp /boost/assign//boost_assign ]
     ;
 
 test-suite phoenix_boost_bind_compatibility :
@@ -217,7 +217,7 @@ test-suite phoenix_boost_bind_compatibility :
 
 test-suite phoenix_bll_compatibility :
     [ run bll_compatibility/algorithm_test.cpp ]
-    [ run bll_compatibility/bind_tests_advanced.cpp ]
+    [ run bll_compatibility/bind_tests_advanced.cpp /boost/any//boost_any ]
 #    [ run bll_compatibility/bind_tests_simple.cpp ]
 #    [ run bll_compatibility/bind_tests_simple_f_refs.cpp ]
 #    [ run bll_compatibility/bll_and_function.cpp ]
@@ -242,7 +242,7 @@ test-suite phoenix_regression :
     [ run regression/bug5626.cpp ]
     [ run regression/bug5824.cpp ]
  #   [ run regression/bug5875.cpp ]
-    [ run regression/bug5968.cpp ]
+    [ run regression/bug5968.cpp /boost/signals2//boost_signals2 ]
     [ run regression/bug6040.cpp ]
     [ run regression/bug6268.cpp ]
     [ run regression/bug7165.cpp ]
