diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 717a163..20cee05 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -24,7 +24,7 @@ run function_test.cpp : : : <rtti>off <toolset>gcc-4.4,<cxxstd>0x:<build>no : fu
 run function_n_test.cpp ;
 run allocator_test.cpp ;
 run stateless_test.cpp ;
-run lambda_test.cpp ;
+run lambda_test.cpp : : : <source>/boost/lambda//boost_lambda ;
 compile-fail function_test_fail1.cpp ;
 compile-fail function_test_fail2.cpp ;
 compile function_30.cpp ;
@@ -42,7 +42,7 @@ run function_ref_portable.cpp ;
 run contains_test.cpp ;
 run contains2_test.cpp ;
 run nothrow_swap.cpp ;
-run rvalues_test.cpp ;
+run rvalues_test.cpp : : : <source>/boost/move//boost_move ;
 compile function_typeof_test.cpp
    : <cxxstd>03:<build>no <cxxstd>98:<build>no <cxxstd>0x:<build>no ;
 run result_arg_types_test.cpp ;
