diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index f12d0eb..b1582f9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -22,13 +22,14 @@ project
         <cxxstd>11
         <threading>multi
     : requirements
+        <source>/boost/config//boost_config
+        <source>/boost/exception//boost_exception
         <define>BOOST_LEAF_BOOST_AVAILABLE
         <threading>single:<define>BOOST_LEAF_NO_THREADS
         <toolset>gcc:<cxxflags>"-Wno-delete-non-virtual-dtor -Wno-parentheses"
         <toolset>clang:<cxxflags>"-Wno-unused-variable -Wno-delete-non-virtual-dtor -Wno-non-virtual-dtor -Wno-dangling-else"
         <toolset>darwin:<cxxflags>"-Wno-unused-variable -Wno-delete-non-virtual-dtor -Wno-non-virtual-dtor -Wno-delete-non-abstract-non-virtual-dtor -Wno-dangling-else"
         <toolset>msvc:<cxxflags>"-wd 4267 -wd 4996 -wd 4244"
-        <include>../../..
     ;
 
 compile _hpp_common_test.cpp ;
