diff --git a/example/Jamfile b/example/Jamfile
index 08ab43d..7af3c18 100644
--- a/example/Jamfile
+++ b/example/Jamfile
@@ -6,14 +6,20 @@
 
 #-------------------------------------------------------------------------------
 
-exe  interpreter_example     : interpreter_example.cpp ;
+exe  interpreter_example     : interpreter_example.cpp
+    : <library>/boost/tokenizer//boost_tokenizer
+      <library>/boost/lexical_cast//boost_lexical_cast
+    ;
 
 exe  result_of_example       : result_of_example.cpp   ;
 
 exe  interface_example       : interface_example.cpp   ;
 
 exe  fast_mem_fn_example     : fast_mem_fn_example.cpp 
-    : <include>. ; # needed for Boost.PP file iteration with some compilers
+    : <include>. # needed for Boost.PP file iteration with some compilers
+      <library>/boost/timer//boost_timer
+      <library>/boost/bind//boost_bind
+    ;
 
 exe  macro_type_args_example : macro_type_args_example.cpp ;
 
diff --git a/test/Jamfile b/test/Jamfile
index e652c46..084dae7 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -75,12 +75,14 @@ import testing ;
     
     # Code from the examples
 
-    [ compile ../example/interpreter_example.cpp ]
+    [ compile ../example/interpreter_example.cpp : <library>/boost/tokenizer//boost_tokenizer <library>/boost/lexical_cast//boost_lexical_cast ]
     [ compile ../example/result_of_example.cpp ]
     [ compile ../example/interface_example.cpp ]
     [ compile ../example/fast_mem_fn_example.cpp 
       # needed for Boost.PP file iteration with some compilers
       : <include>../example
+        <library>/boost/timer//boost_timer
+        <library>/boost/bind//boost_bind
     ]
     [ compile ../example/macro_type_args_example.cpp ]
     ;
