diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index a795194..d3fea29 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -42,7 +42,7 @@ rule check-synchronization-lib ( properties * )
     return $(result) ;
 }
 
-project boost/atomic
+project
     : requirements
       <include>../src
       <threading>multi
@@ -143,5 +143,3 @@ lib boost_atomic
     : usage-requirements
       <conditional>@check-synchronization-lib
    ;
-
-boost-install boost_atomic ;
