diff --git a/example/cpp03/http/server/Jamfile.v2 b/example/cpp03/http/server/Jamfile.v2
index 02364eea..0f21de2e 100644
--- a/example/cpp03/http/server/Jamfile.v2
+++ b/example/cpp03/http/server/Jamfile.v2
@@ -24,6 +24,7 @@ exe server
     /boost/system//boost_system
     /boost/chrono//boost_chrono
     /boost/thread//boost_thread
+    /boost/logic//boost_logic
   : <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/http/server2/Jamfile.v2 b/example/cpp03/http/server2/Jamfile.v2
index fcfd6d91..cd127d41 100644
--- a/example/cpp03/http/server2/Jamfile.v2
+++ b/example/cpp03/http/server2/Jamfile.v2
@@ -24,6 +24,7 @@ exe server
     /boost/system//boost_system
     /boost/chrono//boost_chrono
     /boost/thread//boost_thread
+    /boost/logic//boost_logic
   : <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/http/server3/Jamfile.v2 b/example/cpp03/http/server3/Jamfile.v2
index c1582963..25f14692 100644
--- a/example/cpp03/http/server3/Jamfile.v2
+++ b/example/cpp03/http/server3/Jamfile.v2
@@ -23,6 +23,7 @@ exe server
     /boost/system//boost_system
     /boost/chrono//boost_chrono
     /boost/thread//boost_thread
+    /boost/logic//boost_logic
   : <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/http/server4/Jamfile.v2 b/example/cpp03/http/server4/Jamfile.v2
index 8168162e..3e556436 100644
--- a/example/cpp03/http/server4/Jamfile.v2
+++ b/example/cpp03/http/server4/Jamfile.v2
@@ -22,6 +22,7 @@ exe server
     /boost/system//boost_system
     /boost/chrono//boost_chrono
     /boost/thread//boost_thread
+    /boost/logic//boost_logic
   : <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/porthopper/Jamfile.v2 b/example/cpp03/porthopper/Jamfile.v2
index 7fc6784b..56e45d2b 100644
--- a/example/cpp03/porthopper/Jamfile.v2
+++ b/example/cpp03/porthopper/Jamfile.v2
@@ -16,6 +16,7 @@ project
   : requirements
     <library>/boost/system//boost_system
     <library>/boost/chrono//boost_chrono
+    <library>/boost/lambda//boost_lambda
     <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/spawn/Jamfile.v2 b/example/cpp03/spawn/Jamfile.v2
index 21ce9048..f6c9243b 100644
--- a/example/cpp03/spawn/Jamfile.v2
+++ b/example/cpp03/spawn/Jamfile.v2
@@ -36,6 +36,7 @@ exe parallel_grep
     /boost/context//boost_context
     /boost/coroutine//boost_coroutine
     /boost/system//boost_system
+    /boost/thread//boost_thread
   : <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp03/ssl/Jamfile.v2 b/example/cpp03/ssl/Jamfile.v2
index 03d5862c..da60fa87 100644
--- a/example/cpp03/ssl/Jamfile.v2
+++ b/example/cpp03/ssl/Jamfile.v2
@@ -14,16 +14,7 @@ lib mswsock ; # NT
 lib ipv6 ; # HPUX
 lib network ; # HAIKU
 
-if [ os.name ] = NT
-{
-  lib ssl : : <name>ssleay32 ;
-  lib crypto : : <name>libeay32  ;
-}
-else
-{
-  lib ssl ;
-  lib crypto ;
-}
+using openssl ;
 
 project
   : requirements
@@ -40,8 +31,8 @@ project
     <target-os>hpux,<toolset>gcc:<define>_XOPEN_SOURCE_EXTENDED
     <target-os>hpux:<library>ipv6
     <target-os>haiku:<library>network
-    <library>ssl
-    <library>crypto
+    <library>/openssl//ssl
+    <library>/openssl//crypto
   ;
 
 exe client : client.cpp ;
diff --git a/example/cpp03/timeouts/Jamfile.v2 b/example/cpp03/timeouts/Jamfile.v2
index 76583c85..3f7bb08d 100644
--- a/example/cpp03/timeouts/Jamfile.v2
+++ b/example/cpp03/timeouts/Jamfile.v2
@@ -16,6 +16,7 @@ project
   : requirements
     <library>/boost/system//boost_system
     <library>/boost/chrono//boost_chrono
+    <library>/boost/lambda//boost_lambda
     <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
diff --git a/example/cpp11/files/Jamfile.v2 b/example/cpp11/files/Jamfile.v2
index 28915dfe..514923ca 100644
--- a/example/cpp11/files/Jamfile.v2
+++ b/example/cpp11/files/Jamfile.v2
@@ -5,9 +5,9 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
 
-lib uring ; # LINUX
-lib ws2_32 ; # NT
-lib mswsock ; # NT
+searched-lib uring ; # LINUX
+searched-lib ws2_32 ; # NT
+searched-lib mswsock ; # NT
 
 project
   : requirements
diff --git a/example/cpp11/ssl/Jamfile.v2 b/example/cpp11/ssl/Jamfile.v2
index 03d5862c..da60fa87 100644
--- a/example/cpp11/ssl/Jamfile.v2
+++ b/example/cpp11/ssl/Jamfile.v2
@@ -14,16 +14,7 @@ lib mswsock ; # NT
 lib ipv6 ; # HPUX
 lib network ; # HAIKU
 
-if [ os.name ] = NT
-{
-  lib ssl : : <name>ssleay32 ;
-  lib crypto : : <name>libeay32  ;
-}
-else
-{
-  lib ssl ;
-  lib crypto ;
-}
+using openssl ;
 
 project
   : requirements
@@ -40,8 +31,8 @@ project
     <target-os>hpux,<toolset>gcc:<define>_XOPEN_SOURCE_EXTENDED
     <target-os>hpux:<library>ipv6
     <target-os>haiku:<library>network
-    <library>ssl
-    <library>crypto
+    <library>/openssl//ssl
+    <library>/openssl//crypto
   ;
 
 exe client : client.cpp ;
diff --git a/example/cpp14/parallel_group/Jamfile.v2 b/example/cpp14/parallel_group/Jamfile.v2
index ecf9497e..98d8ad41 100644
--- a/example/cpp14/parallel_group/Jamfile.v2
+++ b/example/cpp14/parallel_group/Jamfile.v2
@@ -16,6 +16,7 @@ project
   : requirements
     <library>/boost/system//boost_system
     <library>/boost/chrono//boost_chrono
+    <library>/boost/thread//boost_thread
     <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
