diff --git a/example/cpp11/files/Jamfile.v2 b/example/cpp11/files/Jamfile.v2
index ac73d533..f8db792b 100644
--- a/example/cpp11/files/Jamfile.v2
+++ b/example/cpp11/files/Jamfile.v2
@@ -5,9 +5,9 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
 
-lib uring ; # LINUX
-lib ws2_32 ; # NT
-lib mswsock ; # NT
+searched-lib uring ; # LINUX
+searched-lib ws2_32 ; # NT
+searched-lib mswsock ; # NT
 
 project
   : requirements
diff --git a/example/cpp11/ssl/Jamfile.v2 b/example/cpp11/ssl/Jamfile.v2
index 2afc3fe1..0747e56a 100644
--- a/example/cpp11/ssl/Jamfile.v2
+++ b/example/cpp11/ssl/Jamfile.v2
@@ -14,16 +14,7 @@ lib mswsock ; # NT
 lib ipv6 ; # HPUX
 lib network ; # HAIKU
 
-if [ os.name ] = NT
-{
-  lib ssl : : <name>ssleay32 ;
-  lib crypto : : <name>libeay32  ;
-}
-else
-{
-  lib ssl ;
-  lib crypto ;
-}
+using openssl ;
 
 project
   : requirements
@@ -40,8 +31,8 @@ project
     <target-os>hpux,<toolset>gcc:<define>_XOPEN_SOURCE_EXTENDED
     <target-os>hpux:<library>ipv6
     <target-os>haiku:<library>network
-    <library>ssl
-    <library>crypto
+    <library>/openssl//ssl
+    <library>/openssl//crypto
   ;
 
 exe client : client.cpp ;
diff --git a/example/cpp14/parallel_group/Jamfile.v2 b/example/cpp14/parallel_group/Jamfile.v2
index 49e452ec..011814db 100644
--- a/example/cpp14/parallel_group/Jamfile.v2
+++ b/example/cpp14/parallel_group/Jamfile.v2
@@ -16,6 +16,7 @@ project
   : requirements
     <library>/boost/system//boost_system
     <library>/boost/chrono//boost_chrono
+    <library>/boost/thread//boost_thread
     <define>BOOST_ALL_NO_LIB=1
     <threading>multi
     <target-os>solaris:<library>socket
