diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index 7233e02d..00000000
--- a/Jamfile
+++ /dev/null
@@ -1,173 +0,0 @@
-#
-# Copyright (c) 2019 Vinnie Falco (vinnie dot falco at gmail dot com)
-#
-# Distributed under the Boost Software License, Version 1.0. (See accompanying
-# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-#
-# Official repository: https://github.com/boostorg/beast
-#
-
-import ac ;
-import os ;
-import path ;
-import feature ;
-import boost ;
-import modules ;
-import testing ;
-import ../config/checks/config : requires ;
-
-boost.use-project ;
-
-lib socket ; # SOLARIS, QNXNTO
-lib nsl ; # SOLARIS
-lib ws2_32 ; # NT
-lib mswsock ; # NT
-lib ipv6 ; # HPUX
-lib network ; # HAIKU
-
-lib ssl ;
-lib crypto ;
-lib crypt32 ;
-
-# Microsoft Windows section. Refer to FAQ "Windows and OpenSSL"
-if [ os.name ] = NT
-{
-  local OPENSSL_ROOT_DEFAULT = "C:/OpenSSL" ;
-  local OPENSSL_ROOT_ENV = [ os.environ OPENSSL_ROOT ] ;
-  local OPENSSL_ROOT = "" ;
-  if $(OPENSSL_ROOT_ENV)
-    {
-       OPENSSL_ROOT = $(OPENSSL_ROOT_ENV) ;
-    }
-  else 
-    {
-      OPENSSL_ROOT = $(OPENSSL_ROOT_DEFAULT) ;
-    }
-  project
-    : requirements
-      <include>$(OPENSSL_ROOT)/include
-      <variant>debug:<library-path>$(OPENSSL_ROOT)/lib
-      <target-os>windows<variant>debug:<library-path>$(OPENSSL_ROOT)/debug/lib
-      <variant>release:<library-path>$(OPENSSL_ROOT)/lib
-    ;
-
-  if [ path.exists $(OPENSSL_ROOT)/lib/libssl.lib ]
-    {
-      echo "OpenSSL > 1.1.0. Including libssl" ;
-      lib ssl : : <target-os>windows <name>libssl ;
-    }
-  if [ path.exists $(OPENSSL_ROOT)/lib/libcrypto.lib ]
-    {
-      echo "OpenSSL > 1.1.0. Including libcrypto" ;
-      lib crypto : : <target-os>windows <name>libcrypto ;
-    }
-  if [ path.exists $(OPENSSL_ROOT)/lib/ssleay32.lib ]
-    {
-      echo "OpenSSL < 1.1.0. Including ssleay32" ;
-      lib ssl : : <target-os>windows <name>ssleay32 ;
-    }
-  if [ path.exists $(OPENSSL_ROOT)/lib/libeay32.lib ]
-    {
-      echo "OpenSSL < 1.1.0. Including libeay32" ;
-      lib crypto : : <target-os>windows <name>libeay32 ;
-    }
-}
-
-feature.feature boost.beast.allow-deprecated : on off : propagated composite ;
-feature.compose <boost.beast.allow-deprecated>on : <define>BOOST_BEAST_ALLOW_DEPRECATED ;
-
-feature.feature boost.beast.separate-compilation : on off : propagated composite ;
-feature.compose <boost.beast.separate-compilation>on : <define>BOOST_BEAST_SEPARATE_COMPILATION ;
-
-variant beast_coverage
-  : debug
-  : <cxxflags>"-msse4.2 --coverage"
-    <linkflags>"--coverage"
-  ;
-
-variant beast_valgrind
-  : release
-  : <valgrind>on # Boost.Context uses this property
-  ;
-
-variant beast_ubasan
-  : release
-  : <cxxflags>"-msse4.2 -funsigned-char -fno-omit-frame-pointer -fsanitize=address,undefined -fno-sanitize-recover=address,undefined -fsanitize-blacklist=libs/beast/tools/blacklist.supp"
-    <linkflags>"-fsanitize=address,undefined"
-    <define>BOOST_USE_ASAN=1
-  ;
-
-path-constant TEST_MAIN : include/boost/beast/_experimental/unit_test/main.cpp ;
-
-project /boost/beast ;
-
-local defines =
-    [ requires
-    cxx11_constexpr
-    cxx11_decltype
-    cxx11_hdr_tuple
-    cxx11_template_aliases
-    cxx11_variadic_templates
-    ]
-    <implicit-dependency>/boost//headers
-    <include>.
-    <define>BOOST_ALL_NO_LIB=1
-    <define>BOOST_ASIO_SEPARATE_COMPILATION
-    <define>BOOST_ASIO_NO_DEPRECATED=1
-    <define>BOOST_ASIO_DISABLE_BOOST_ARRAY=1
-    <define>BOOST_ASIO_DISABLE_BOOST_BIND=1
-    <define>BOOST_ASIO_DISABLE_BOOST_DATE_TIME=1
-    <define>BOOST_ASIO_DISABLE_BOOST_REGEX=1
-    <define>BOOST_COROUTINES_NO_DEPRECATION_WARNING=1
-    <toolset>msvc:<cxxflags>"/bigobj"
-    <toolset>msvc-14.1:<cxxflags>"/permissive-"
-    <toolset>msvc-14.2:<cxxflags>"/permissive-"
-    <toolset>msvc:<define>_SCL_SECURE_NO_WARNINGS=1
-    <toolset>msvc:<define>_CRT_SECURE_NO_WARNINGS=1
-    <toolset>msvc:<define>_SILENCE_CXX17_ALLOCATOR_VOID_DEPRECATION_WARNING
-    <toolset>msvc:<define>_SILENCE_CXX17_ADAPTOR_TYPEDEFS_DEPRECATION_WARNING
-    <toolset>msvc,<variant>release:<cxxflags>"/Ob2 /Oi /Ot"
-    <target-os>linux:<define>_XOPEN_SOURCE=600
-    <target-os>linux:<define>_GNU_SOURCE=1
-    <target-os>solaris:<define>_XOPEN_SOURCE=500
-    <target-os>solaris:<define>__EXTENSIONS__
-    <target-os>solaris:<library>socket
-    <target-os>solaris:<library>nsl
-    <target-os>windows:<define>_WIN32_WINNT=0x0601
-    <target-os>windows,<toolset>gcc:<library>ws2_32
-    <target-os>windows,<toolset>gcc:<library>mswsock
-    <target-os>windows,<toolset>gcc-cygwin:<define>__USE_W32_SOCKETS
-    <target-os>hpux,<toolset>gcc:<define>_XOPEN_SOURCE_EXTENDED
-    <target-os>hpux:<library>ipv6
-    <target-os>qnxnto:<library>socket
-    <target-os>haiku:<library>network
-    ;
-
-lib lib-asio
-    : test/lib_asio.cpp
-    : requirements
-        <link>static
-        $(defines)
-    : usage-requirements
-        $(defines)
-    ;
-
-lib lib-asio-ssl
-    : test/lib_asio_ssl.cpp
-    : requirements
-        <link>static
-        $(defines)
-        [ ac.check-library /boost/beast//ssl : <library>/boost/beast//ssl/<link>shared : <build>no ]
-        [ ac.check-library /boost/beast//crypto : <library>/boost/beast//crypto/<link>shared : <build>no ]
-    : usage-requirements
-        $(defines)
-    ;
-
-lib lib-beast
-    : test/lib_beast.cpp
-    : requirements
-    <link>static
-    $(defines)
-    : usage-requirements
-    $(defines)
-    ;
diff --git a/example/Jamfile b/example/Jamfile
index 8b49d5fd..c7998722 100644
--- a/example/Jamfile
+++ b/example/Jamfile
@@ -8,7 +8,7 @@
 #
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project /boost/beast/example
     : requirements
@@ -22,6 +22,7 @@ project /boost/beast/example
     ]
     <library>/boost/beast//lib-asio/<link>static
     <boost.beast.separate-compilation>on:<library>/boost/beast//lib-beast/<link>static
+    <include>..
     ;
 
 build-project advanced ;
diff --git a/example/http/client/body/Jamfile b/example/http/client/body/Jamfile
index 3b2f249c..db8caabd 100644
--- a/example/http/client/body/Jamfile
+++ b/example/http/client/body/Jamfile
@@ -11,4 +11,5 @@ exe json_client : json_client.cpp
     :
     <variant>coverage:<build>no
     <variant>ubasan:<build>no
+    <library>/boost/json//boost_json
     ;
diff --git a/test/Jamfile b/test/Jamfile
index 851f8df1..3fa2cab3 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -8,16 +8,14 @@
 #
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 lib lib-test
     : lib_test.cpp
     : requirements
     <link>static
-    $(defines)
     : usage-requirements
-    <include>./extras/include
-    $(defines)
+    <include>extras/include
     ;
 
 project /boost/beast/test
@@ -30,13 +28,14 @@ project /boost/beast/test
         cxx11_template_aliases
         cxx11_variadic_templates
     ]
-    <include>./extern
+    <include>extern
     <define>BOOST_BEAST_TESTS
     <target-os>darwin:<define>Z_HAVE_UNISTD_H=1
     <library>/boost/beast//lib-asio/<link>static
     <boost.beast.separate-compilation>on:<library>/boost/beast//lib-beast/<link>static
     <library>/boost/filesystem//boost_filesystem
     <library>/boost/context//boost_context
+    <include>..
     ;
 
 path-constant ZLIB_SOURCES :
