diff --git a/index/example/Jamfile b/index/example/Jamfile
index 3c7b5b2cd..cd296af33 100644
--- a/index/example/Jamfile
+++ b/index/example/Jamfile
@@ -44,11 +44,11 @@ if $(GLUT_ROOT)
 }
 
 exe random_test : random_test.cpp ;
-link serialize.cpp /boost//serialization : ;
-link benchmark.cpp /boost//chrono : <threading>multi ;
-link benchmark2.cpp /boost//chrono : <threading>multi ;
-link benchmark3.cpp /boost//chrono : <threading>multi ;
-link benchmark_experimental.cpp  /boost//chrono : <threading>multi ;
+link serialize.cpp /boost/serialization//boost_serialization : ;
+link benchmark.cpp /boost/chrono//boost_chrono : <threading>multi ;
+link benchmark2.cpp /boost/chrono//boost_chrono : <threading>multi ;
+link benchmark3.cpp /boost/chrono//boost_chrono : <threading>multi ;
+link benchmark_experimental.cpp  /boost/chrono//boost_chrono : <threading>multi ;
 if $(GLUT_ROOT)
 {
     link glut_vis.cpp glut ;
