diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index ba92b48d..00000000
--- a/Jamfile
+++ /dev/null
@@ -1,11 +0,0 @@
-#  Boost.MPL Library Jamfile
-#
-#  Copyright (c) 2018 James E. King III
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project example ;
-build-project test ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 5761c6d6..ad2d083a 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -53,7 +53,7 @@ test-suite mpl :
     [ compile find.cpp ]
     [ compile find_if.cpp ]
     [ compile fold.cpp ]
-    [ run for_each.cpp ]
+    [ run for_each.cpp : : : <source>/boost/bind//boost_bind ]
     [ compile front.cpp ]
     [ compile get_tag_def.cpp ]
     [ compile has_xxx.cpp ]
