diff --git a/test/Jamfile b/test/Jamfile
index e3ad4e6..2c66296 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -12,6 +12,10 @@ import ../../config/checks/config : requires ;
 
 project
     : requirements
+      <source>/boost/align//boost_align
+      <source>/boost/container_hash//boost_container_hash
+      <source>/boost/functional//boost_functional
+      <source>/boost/smart_ptr//boost_smart_ptr
       <toolset>gcc:<cxxflags>-Wno-non-virtual-dtor
       <toolset>gcc:<cxxflags>-Wno-deprecated-declarations
       #<toolset>gcc:<cxxflags>-Wno-delete-non-virtual-dtor (not in 4.4/4.6)
@@ -79,7 +83,7 @@ run shared_ptr_convertible_test.cpp ;
 run wp_convertible_test.cpp ;
 run ip_convertible_test.cpp ;
 run allocate_shared_test.cpp ;
-run sp_atomic_test.cpp ;
+run sp_atomic_test.cpp : : : <source>/boost/atomic//boost_atomic ;
 run esft_void_test.cpp ;
 run esft_second_ptr_test.cpp ;
 run make_shared_esft_test.cpp ;
@@ -251,7 +255,7 @@ compile make_shared_msvc_test.cpp ;
 
 compile lwm_win32_cs_test.cpp ;
 
-run atomic_sp_test.cpp ;
+run atomic_sp_test.cpp : : : <source>/boost/atomic//boost_atomic ;
 
 run sp_constexpr_test.cpp ;
 run sp_constexpr_test2.cpp ;
@@ -406,19 +410,19 @@ run lsp_hash_test.cpp
 run lsp_hash_test2.cpp ;
 
 run atomic_count_mt_test.cpp
-  : : : <threading>multi ;
+  : : : <threading>multi <source>/boost/bind//boost_bind ;
 
 run spinlock_mt_test.cpp
-  : : : <threading>multi ;
+  : : : <threading>multi <source>/boost/bind//boost_bind ;
 
 run spinlock_pool_mt_test.cpp
-  : : : <threading>multi ;
+  : : : <threading>multi <source>/boost/bind//boost_bind ;
 
 run shared_ptr_mt_test.cpp
-  : : : <threading>multi ;
+  : : : <threading>multi <source>/boost/bind//boost_bind ;
 
 run weak_ptr_mt_test.cpp
-  : : : <threading>multi ;
+  : : : <threading>multi <source>/boost/bind//boost_bind ;
 
 compile sp_report_implementation.cpp ;
 
