diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index bc354e0..55c956c 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -9,6 +9,11 @@
 #
 import testing ;
 
+project : requirements
+    <source>/boost/container//boost_container
+    <source>/boost/type_traits//boost_type_traits
+    ;
+
 rule test_all
 {
    local all_rules = ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 2ce67bb..a588f3a 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -9,6 +9,11 @@
 
 import testing ;
 
+project : requirements
+    <source>/boost/core//boost_core
+    <source>/boost/container//boost_container
+    ;
+
 rule test_all
 {
    local all_rules = ;
