diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index b65ddb2..bc7de84 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -9,12 +9,12 @@ import feature ;
 import modules ;
 import testing ;
 import toolset ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 feature.feature numa : on : optional propagated composite ;
 feature.compose <numa>on : <define>BOOST_USE_NUMA ;
 
-project boost/fiber
+project
     : requirements
       <library>/boost/context//boost_context
       <library>/boost/filesystem//boost_filesystem
@@ -65,7 +65,7 @@ lib boost_fiber
       recursive_timed_mutex.cpp
       timed_mutex.cpp
       scheduler.cpp
-    : <link>shared:<library>../../context/build//boost_context
+    : <link>shared:<library>/boost/context//boost_context
     [ requires cxx11_auto_declarations
                cxx11_constexpr
                cxx11_defaulted_functions
@@ -145,6 +145,3 @@ lib boost_fiber_numa
                cxx11_thread_local
                cxx11_variadic_templates ]
     ;
-
-
-boost-install boost_fiber boost_fiber_numa ;
diff --git a/examples/Jamfile.v2 b/examples/Jamfile.v2
index 75dddf1..0657eb2 100644
--- a/examples/Jamfile.v2
+++ b/examples/Jamfile.v2
@@ -14,7 +14,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/fiber/example
+project
     : requirements
       <library>../build//boost_fiber
       <library>/boost/context//boost_context
@@ -42,12 +42,12 @@ exe range_for : range_for.cpp ;
 exe priority : priority.cpp ;
 exe segmented_stack : segmented_stack.cpp ;
 exe simple : simple.cpp ;
-exe wait_stuff : wait_stuff.cpp ;
+exe wait_stuff : wait_stuff.cpp /boost/variant//boost_variant ;
 exe work_sharing : work_sharing.cpp ;
 exe work_stealing : work_stealing.cpp ;
 
-exe asio/autoecho : asio/autoecho.cpp ;
-exe asio/exchange : asio/exchange.cpp ;
-exe asio/ps/publisher : asio/ps/publisher.cpp ;
-exe asio/ps/server : asio/ps/server.cpp ;
-exe asio/ps/subscriber : asio/ps/subscriber.cpp ;
+exe asio/autoecho : asio/autoecho.cpp /boost/asio//boost_asio ;
+exe asio/exchange : asio/exchange.cpp /boost/asio//boost_asio ;
+exe asio/ps/publisher : asio/ps/publisher.cpp /boost/asio//boost_asio ;
+exe asio/ps/server : asio/ps/server.cpp /boost/asio//boost_asio ;
+exe asio/ps/subscriber : asio/ps/subscriber.cpp /boost/asio//boost_asio ;
diff --git a/examples/numa/Jamfile.v2 b/examples/numa/Jamfile.v2
index 15420cb..ae5430e 100644
--- a/examples/numa/Jamfile.v2
+++ b/examples/numa/Jamfile.v2
@@ -14,7 +14,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/fiber/example/numa
+project
     : requirements
       <library>../../build//boost_fiber
       <library>../../build//boost_fiber_numa
diff --git a/performance/fiber/Jamfile.v2 b/performance/fiber/Jamfile.v2
index 4727be3..b5ba8aa 100644
--- a/performance/fiber/Jamfile.v2
+++ b/performance/fiber/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/fiber/performance/fiber
+project
     : requirements
       <library>/boost/fiber//boost_fiber
       <library>/boost/fiber//boost_fiber_numa
diff --git a/performance/fiber/numa/Jamfile.v2 b/performance/fiber/numa/Jamfile.v2
index 30f3a7f..f6c6c84 100644
--- a/performance/fiber/numa/Jamfile.v2
+++ b/performance/fiber/numa/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/fiber/performance/fiber/numa
+project
     : requirements
       <library>/boost/fiber//boost_fiber
       <target-os>solaris:<linkflags>"-llgrp"
diff --git a/performance/thread/Jamfile.v2 b/performance/thread/Jamfile.v2
index baa1821..bc612e3 100644
--- a/performance/thread/Jamfile.v2
+++ b/performance/thread/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/fiber/performance/thread
+project
     : requirements
       <link>static
       <threading>multi
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 4e45022..359d68f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,11 +13,11 @@ import os ;
 import path ;
 import testing ;
 import toolset ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
-project boost/fiber/test
+project 
     : requirements
-      <library>../../test/build//boost_unit_test_framework
+      <library>/boost/test//boost_unit_test_framework
       <library>/boost/context//boost_context
       <library>/boost/fiber//boost_fiber
       <library>/boost/thread//boost_thread
