diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index fbd332f..26cf0b0 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,7 +7,12 @@
 # Home at http://www.boost.org/libs/scope_exit
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
+
+project : requirements
+    <source>/boost/foreach//boost_foreach
+    <source>/boost/rational//boost_rational
+    ;
 
 rule run-vaseq ( target : requirements * )
 {
