diff --git a/bench/Jamfile b/bench/Jamfile
index 3057fb9..39aa7d4 100644
--- a/bench/Jamfile
+++ b/bench/Jamfile
@@ -11,7 +11,7 @@ import common ;
 import os ;
 import path ;
 import property ;
-
+import notfile ;
 import feature ;
 
 path-constant HERE : . ;
diff --git a/build/Jamfile b/build/Jamfile
index 3eb4d55..532e08d 100644
--- a/build/Jamfile
+++ b/build/Jamfile
@@ -9,7 +9,7 @@
 #
 
 import modules ;
-import ../../config/checks/config ;
+import config ;
 
 # These make sure we only build on compatible C++11 or later toolchains.
 obj cxx11_basic_alignas : ../check/basic_alignas.cpp ;
@@ -54,7 +54,7 @@ rule warn-if-requested ( properties * )
     }
 }
 
-project boost/json
+project
     : requirements
       [ requires
         config/cxx11_constexpr
@@ -79,13 +79,13 @@ project boost/json
 
 alias json_deps
    : usage-requirements
-     <library>/boost//container/<warnings-as-errors>off
+     <library>/boost/container//boost_container/<warnings-as-errors>off
    ;
 
 alias json_sources
    : src.cpp
    : usage-requirements
-     <library>/boost//container/<warnings-as-errors>off
+     <library>/boost/container//boost_container/<warnings-as-errors>off
    ;
 
 explicit json_deps json_sources ;
@@ -93,7 +93,5 @@ explicit json_deps json_sources ;
 lib boost_json
    : json_sources
    : usage-requirements
-     <library>/boost//container/<warnings-as-errors>off
+     <library>/boost/container//boost_container/<warnings-as-errors>off
    ;
-
-boost-install boost_json ;
