diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 068ef6d..f5ba6f9 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -18,7 +18,7 @@ project :
 run add_cxx11_lambda.cpp ;
 run add_global_functor.cpp ;
 run add_local_functor.cpp ;
-run add_phoenix.cpp ;
+run add_phoenix.cpp : : : <source>/boost/spirit//boost_spirit ;
 
 run const_block.cpp ;
 compile-fail const_block_error.cpp : <variant>debug ;
@@ -43,8 +43,8 @@ run n2550_find_if.cpp ;
 compile-fail noncopyable_cxx11_lambda_error.cpp ;
 run noncopyable_local_function.cpp ;
 
-run phoenix_factorial.cpp ;
-run phoenix_factorial_local.cpp ;
+run phoenix_factorial.cpp : : : <source>/boost/phoenix//boost_phoenix ;
+run phoenix_factorial_local.cpp : : : <source>/boost/phoenix//boost_phoenix ;
 
 # Only compile but do not run profiling programs (they take a long time to run).
 exe profile_global_functor : profile_global_functor.cpp
@@ -75,8 +75,9 @@ exe profile_local_functor : profile_local_functor.cpp
 exe profile_phoenix : profile_phoenix.cpp
     :   <library>/boost/chrono//boost_chrono
         <library>/boost/system//boost_system
+        <source>/boost/phoenix//boost_phoenix
         <link>static
     ;
 
-run scope_exit.cpp ;
+run scope_exit.cpp : : : <source>/boost/foreach//boost_foreach ;
 
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 20bed34..8cda65f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,6 +13,8 @@ project :
     <toolset>sun:<define>BOOST_TYPEOF_EMULATION
     <toolset>gcc:<cxxflags>-Wno-unused-local-typedefs
     <toolset>clang:<cxxflags>-Wno-unused-local-typedefs
+    <source>/boost/concept_check//boost_concept_check
+    <source>/boost/functional//boost_functional
     ;
 
 rule vaseq ( command target )
