diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 852054d..a90b51c 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -1,5 +1,5 @@
 
-project boost/program_options 
+project 
     : source-location ../src 
     ;
 
@@ -9,8 +9,9 @@ SOURCES =
     convert winmain split
     ;
 
-boost-lib program_options
+lib boost_program_options
     : $(SOURCES).cpp
     : # See https://svn.boost.org/trac/boost/ticket/5049
       <target-os>hpux,<toolset>gcc:<define>_INCLUDE_STDC__SOURCE_199901
-    ;
\ No newline at end of file
+      <link>shared:<define>BOOST_PROGRAM_OPTIONS_DYN_LINK=1
+    ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 0f1a6e7..968b5a8 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -38,7 +38,7 @@ test-suite program_options :
     [ run options_description_test.cpp : : : <rtti>off : options_description_no_rtti_test ]
     ;
         
-exe test_convert : test_convert.cpp ;   
+exe test_convert : test_convert.cpp /boost/timer//boost_timer ;
 
 # `quick` target (for CI)
 run quick.cpp : --path=initial ;
