diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 3e0b8fc..e027fa5 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -12,7 +12,7 @@ import config : requires ;
 
 local cxx11-requirements = [ requires cxx11_hdr_ratio cxx11_template_aliases cxx11_decltype cxx11_char16_t cxx11_char32_t ] ;
 
-project boost/chrono
+project
     : source-location ../src
     : requirements
         <target-os>freebsd:<linkflags>"-lrt"
@@ -111,5 +111,3 @@ lib boost_chrono
     <link>shared:<define>BOOST_CHRONO_DYN_LINK=1 # tell source we're building dll's
     <link>static:<define>BOOST_CHRONO_STATIC_LINK=1 # tell source we're building static lib's
     ;
-
-boost-install boost_chrono ;
