diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 9ee4d4e..b631e7e 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,11 +5,12 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+import config : requires ;
 import testing ;
 import feature ;
 
-import ../../config/checks/config : requires ;
-
 project
     : requirements
         [ requires cxx11_decltype ]
