diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 3894232..09832f2 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -4,7 +4,7 @@
 # file LICENSE_1_0.txt or a copy at http://www.boost.org/LICENSE_1_0.txt).
 # See: http://www.boost.org/doc/libs/release/libs/contract/doc/html/index.html
 
-project boost/contract
+project
     : source-location ../src
     : requirements
         <link>shared:<define>BOOST_CONTRACT_DYN_LINK
@@ -19,5 +19,3 @@ project boost/contract
 # If lib as header-only, none of following will be used.
 lib boost_contract : contract.cpp : <link>shared ;
 lib boost_contract : contract.cpp : <link>static ;
-
-boost-install boost_contract ;
diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 3fe6e2f..bd4f050 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -4,8 +4,10 @@
 # file LICENSE_1_0.txt or a copy at http://www.boost.org/LICENSE_1_0.txt).
 # See: http://www.boost.org/doc/libs/release/libs/contract/doc/html/index.html
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
 import ../build/boost_contract_build ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 test-suite features :
     [ boost_contract_build.subdir-run features : introduction_comments ]
@@ -54,7 +56,8 @@ test-suite features :
     [ boost_contract_build.subdir-run-cxx11 features : old_no_macro ]
     # Still needs C++11 for OLDOF variadic macros.
     [ boost_contract_build.subdir-run-cxx11 features : no_lambdas ]
-    [ boost_contract_build.subdir-run-cxx11 features : no_lambdas_local_func ]
+    [ boost_contract_build.subdir-run-cxx11 features : no_lambdas_local_func :
+        <source>/boost/local_function//boost_local_function ]
 ;
 
 test-suite n1962 :
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 2026b8d..564f5c9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -4,8 +4,15 @@
 # file LICENSE_1_0.txt or a copy at http://www.boost.org/LICENSE_1_0.txt).
 # See: http://www.boost.org/doc/libs/release/libs/contract/doc/html/index.html
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
 import ../build/boost_contract_build ;
-import ../../config/checks/config : requires ;
+import config : requires ;
+
+project
+    : requirements
+        <source>/boost/iostreams//boost_iostreams
+    ;
 
 test-suite constructor :
     [ boost_contract_build.subdir-run-cxx11 constructor : smoke ]
