diff --git a/example/Jamfile b/example/Jamfile
index 6b4b0f3..8093ca9 100644
--- a/example/Jamfile
+++ b/example/Jamfile
@@ -9,7 +9,8 @@
 
 exe timing_tests 
     : timing_tests.cpp 
-      ../../timer/build//boost_timer
+      /boost/timer//boost_timer
+      /boost/detail//boost_detail
     ;
 
 exe example1
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 91f91de..4e38eac 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -9,13 +9,16 @@
 import os ;
 
 # import rules for testing conditional on config file variables
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 test-suite dynamic_bitset :
 
-    [ run dyn_bitset_unit_tests1.cpp : : : <library>/boost/system//boost_system ]
-    [ run dyn_bitset_unit_tests2.cpp : : : <library>/boost/system//boost_system ]
-    [ run dyn_bitset_unit_tests3.cpp : : : <library>/boost/system//boost_system ]
+    [ run dyn_bitset_unit_tests1.cpp : : : <library>/boost/filesystem//boost_filesystem
+                                           <library>/boost/system//boost_system ]
+    [ run dyn_bitset_unit_tests2.cpp : : : <library>/boost/filesystem//boost_filesystem
+                                           <library>/boost/system//boost_system ]
+    [ run dyn_bitset_unit_tests3.cpp : : : <library>/boost/filesystem//boost_filesystem
+                                           <library>/boost/system//boost_system ]
     [ run dyn_bitset_unit_tests4.cpp : : : <library>/boost/filesystem//boost_filesystem
                                            <library>/boost/system//boost_system ]
     [ run test_ambiguous_set.cpp ]
@@ -36,6 +39,7 @@ if ! [ os.environ UBSAN_OPTIONS ]
 
     [ run dyn_bitset_unit_tests5.cpp
        : : : <define>_SCL_SECURE_NO_WARNINGS=1
+             <library>/boost/filesystem//boost_filesystem
              <library>/boost/serialization//boost_serialization
              <library>/boost/system//boost_system ]
     ;
