diff --git a/Jamfile.v2 b/Jamfile.v2
deleted file mode 100644
index cc332e11..00000000
--- a/Jamfile.v2
+++ /dev/null
@@ -1,9 +0,0 @@
-#  (C) Copyright Juergen Hunold 2006-2010.
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-#
-#  See http://www.boost.org/libs/test for the library home page.
-
-build-project example ;
-build-project test ;
diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index c4fa6e0c..4831cb28 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -6,11 +6,11 @@
 
 import os ;
 
-import ../../predef/check/predef
+import predef
     : check
     : predef-check ;
 
-project boost/test
+project
     : source-location ../src
     : requirements <link>shared:<define>BOOST_TEST_DYN_LINK=1
                    <toolset>borland:<cxxflags>-w-8080
@@ -114,6 +114,6 @@ alias minimal ;
 
 alias included ;
 
-boost-install boost_prg_exec_monitor
-              boost_test_exec_monitor
-              boost_unit_test_framework ;
+# boost-install boost_prg_exec_monitor
+#               boost_test_exec_monitor
+#               boost_unit_test_framework ;
diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 5afb37e0..cd71c0c1 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -11,7 +11,7 @@ import testing ;
 import os ;
 
 # requirements
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 local HAS_UBSAN = "NO_UBSAN" ; # need to defined something
 if [ os.environ UBSAN ]
@@ -22,7 +22,7 @@ if [ os.environ UBSAN ]
 ECHO $(HAS_UBSAN:J) ;
 
 # Project
-project boost/test-examples
+project
     :
     : requirements
       <toolset>clang:<cxxflags>-Wno-c99-extensions <define>$(HAS_UBSAN:J)
@@ -31,12 +31,12 @@ project boost/test-examples
 # Define aliases for the needed libs to get shorter names
 alias prg_exec_monitor
     : # sources
-        /boost//prg_exec_monitor
+        /boost/test//boost_prg_exec_monitor
     ;
 
 alias unit_test_framework
     : # sources
-        /boost//unit_test_framework
+        /boost/test//boost_unit_test_framework
     ;
 
 # make aliases explicit so the libraries will only be built when requested
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 513b7414..c09e8633 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -12,13 +12,13 @@ import property-set ;
 import regex ;
 import "class" : new ;
 
-import ../../config/checks/config : requires ;
+import config : requires ;
 
-project boost/test-tests
+project
     : requirements
     ;
 
-import ../../predef/check/predef
+import predef
     : check require
     : predef-check predef-require ;
 
@@ -102,6 +102,7 @@ rule docs-example-as-test ( test-file )
               :
               :
               : $(requirements_documentation) # requirements
+                <source>/boost/rational//boost_rational
             ] ;
 }
 
@@ -336,19 +337,19 @@ rule boost.test-smoke-ts-logger ( test-name-prefix : logger ? : log_or_report ?
    return $(to-return) ;
 }
 
-exe smoke-ts-included : smoke-ts/basic-smoke-test.cpp
+alias smoke-ts-included : smoke-ts/basic-smoke-test.cpp
                       : $(requirements_datasets) ;
 
-exe smoke-ts-included-2 : smoke-ts/basic-smoke-test2.cpp ;
+alias smoke-ts-included-2 : smoke-ts/basic-smoke-test2.cpp ;
 
-exe smoke-ts-included-3 : smoke-ts/basic-smoke-test3.cpp ;
+alias smoke-ts-included-3 : smoke-ts/basic-smoke-test3.cpp ;
 
 # for template test case filtering from the command line
-exe smoke-ts-included-4 : smoke-ts/basic-smoke-test4.cpp ;
+alias smoke-ts-included-4 : smoke-ts/basic-smoke-test4.cpp ;
 
-exe check-streams-on-exit : framework-ts/check-streams-on-exit.cpp ;
+alias check-streams-on-exit : framework-ts/check-streams-on-exit.cpp ;
 
-exe dataset-master-test-suite-accessible-test : test-organization-ts/dataset-master-test-suite-accessible-test.cpp
+alias dataset-master-test-suite-accessible-test : test-organization-ts/dataset-master-test-suite-accessible-test.cpp
                                               : $(requirements_datasets) ;
 
 alias "smoke-ts"
@@ -434,12 +435,12 @@ alias "smoke-ts"
 [ run smoke-ts-included-4 : \"--run_test=some_suite/test<my_struct<float_ float>>,test<my_struct<char_ char>>\" \"--run_test=test<my_struct<int_ int>>\" : : : cla-template-test-case-sanity-15 ]
 ;
 
-exe custom-command-line-binary-1 : ../doc/examples/runtime-configuration_1.run-fail.cpp ;
-exe custom-command-line-binary-2 : ../doc/examples/runtime-configuration_2.run-fail.cpp
+alias custom-command-line-binary-1 : ../doc/examples/runtime-configuration_1.run-fail.cpp ;
+alias custom-command-line-binary-2 : ../doc/examples/runtime-configuration_2.run-fail.cpp
                                  : $(requirements_boost_test_full_support) ;
-exe custom-command-line-binary-3 : ../doc/examples/runtime-configuration_3.run-fail.cpp
+alias custom-command-line-binary-3 : ../doc/examples/runtime-configuration_3.run-fail.cpp
                                  : $(requirements_boost_test_full_support) ;
-exe custom-command-line-binary-4 : ../doc/examples/runtime-configuration_4.run-fail.cpp
+alias custom-command-line-binary-4 : ../doc/examples/runtime-configuration_4.run-fail.cpp
                                  : $(requirements_datasets) [ requires cxx11_trailing_result_types cxx11_auto_declarations ] $(l_gcc_c11_rvalue_full_support) ;
 
 alias "custom-command-line-ts"
