diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 7172745..cce3be9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -14,7 +14,7 @@ import regex ;
 rule assign-test ( name )
 {
     return [
-        run $(name).cpp /boost/test//boost_unit_test_framework/<link>static ]
+        run $(name).cpp /boost/test//boost_unit_test_framework/<link>static /boost/multi_index//boost_multi_index ]
         ;
 }
 
@@ -27,10 +27,9 @@ rule test_headers
 {
     local all_rules = ;
     local file ;
-    local headers_path = [ path.make $(BOOST_ROOT)/libs/assign/include/boost/assign ] ;
-    for file in [ path.glob-tree $(headers_path) : *.hpp : assign ]
+    for file in [ glob-tree-ex ../include/boost/assign : *.hpp : assign ]
     {
-        local rel_file = [ path.relative-to $(headers_path) $(file) ] ;
+        local rel_file = [ path.relative-to ../include/boost/assign $(file) ] ;
         # Note: The test name starts with '~' in order to group these tests in the test report table, preferably at the end.
         #       All '/' are replaced with '-' because apparently test scripts have a problem with test names containing slashes.
         local test_name = [ regex.replace $(rel_file) "/" "-" ] ;
@@ -39,10 +38,9 @@ rule test_headers
         all_rules += [ compile compile/decl_header.cpp : <define>"BOOST_ASSIGN_TEST_HEADER=$(rel_file)" <dependency>$(file) : $(decl_test_name) ] ;
     }
 
-    local tests_path = [ path.make $(BOOST_ROOT)/libs/assign/test/compile-fail ] ;
-    for file in [ path.glob-tree $(tests_path) : *.cpp ]
+    for file in [ glob-tree-ex compile-fail : *.cpp ]
     {
-        local rel_file = [ path.relative-to $(tests_path) $(file) ] ;
+        local rel_file = [ path.relative-to compile-fail $(file) ] ;
         local test_name = [ regex.replace [ regex.replace $(rel_file) "/" "-" ] ".cpp" "" ] ;
         local decl_test_name = cf-$(test_name) ;
         # ECHO $(rel_file) ;
