diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index b7268c5..d940304 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -15,7 +15,7 @@ import toolset ;
 
 path-constant TOP : .. ;
 
-project /boost/locale
+project
     : source-location $(TOP)/src/boost/locale
     ;
 
@@ -46,7 +46,7 @@ explicit has_iconv ;
 ICONV_PATH = [ modules.peek : ICONV_PATH ] ;
 
 # There may also be an external iconv library
-lib iconv :
+searched-lib iconv :
   : <search>$(ICONV_PATH)/lib <link>shared <runtime-link>shared
   : : <include>$(ICONV_PATH)/include
   ;
@@ -127,54 +127,54 @@ if [ modules.peek : ICU_ICUIN_NAME ]
 
 if $(ICU_ICUUC_NAME)
 {
-    lib icuuc : : <name>$(ICU_ICUUC_NAME) <conditional>@path_options ;
+    searched-lib icuuc : : <name>$(ICU_ICUUC_NAME) <conditional>@path_options ;
     debug-message Using "$(ICU_ICUUC_NAME)" for library "icuuc" ;
 }
 else
 {
-    lib icuuc : :                                                               <runtime-link>shared <conditional>@path_options ;
-    lib icuuc : : <toolset>msvc                     <variant>debug <name>icuucd <runtime-link>shared <conditional>@path_options ;
-    lib icuuc : : <toolset>intel <target-os>windows <variant>debug <name>icuucd <runtime-link>shared <conditional>@path_options ;
-    lib icuuc : :                                                  <name>sicuuc <runtime-link>static <conditional>@path_options ;
-    lib icuuc : : <toolset>msvc                     <variant>debug <name>sicuucd <runtime-link>static <conditional>@path_options ;
-    lib icuuc : : <toolset>intel <target-os>windows <variant>debug <name>sicuucd <runtime-link>static <conditional>@path_options ;
-    lib icuuc : : <name>this_is_an_invalid_library_name ;
+    searched-lib icuuc : :                                                               <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuuc : : <toolset>msvc                     <variant>debug <name>icuucd <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuuc : : <toolset>intel <target-os>windows <variant>debug <name>icuucd <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuuc : :                                                  <name>sicuuc <runtime-link>static <conditional>@path_options ;
+    searched-lib icuuc : : <toolset>msvc                     <variant>debug <name>sicuucd <runtime-link>static <conditional>@path_options ;
+    searched-lib icuuc : : <toolset>intel <target-os>windows <variant>debug <name>sicuucd <runtime-link>static <conditional>@path_options ;
+    searched-lib icuuc : : <name>this_is_an_invalid_library_name ;
 }
 
 if $(ICU_ICUDT_NAME)
 {
-    lib icudt : : <name>$(ICU_ICUDT_NAME) <conditional>@path_options ;
+    searched-lib icudt : : <name>$(ICU_ICUDT_NAME) <conditional>@path_options ;
     debug-message Using "$(ICU_ICUDT_NAME)" for library "icudt" ;
 }
 else
 {
-    lib icudt : : <name>icudata                                   <runtime-link>shared <conditional>@path_options ;
-    lib icudt : : <name>icudt   <toolset>msvc                     <runtime-link>shared <conditional>@path_options ;
-    lib icudt : : <name>icudt   <toolset>intel <target-os>windows <runtime-link>shared <conditional>@path_options ;
-    lib icudt : : <name>sicudata                                   <runtime-link>static <conditional>@path_options ;
-    lib icudt : : <name>sicudt   <toolset>msvc                     <runtime-link>static <conditional>@path_options ;
-    lib icudt : : <name>sicudt   <toolset>intel <target-os>windows <runtime-link>static <conditional>@path_options ;
-    lib icudt : : <name>this_is_an_invalid_library_name ;
+    searched-lib icudt : : <name>icudata                                   <runtime-link>shared <conditional>@path_options ;
+    searched-lib icudt : : <name>icudt   <toolset>msvc                     <runtime-link>shared <conditional>@path_options ;
+    searched-lib icudt : : <name>icudt   <toolset>intel <target-os>windows <runtime-link>shared <conditional>@path_options ;
+    searched-lib icudt : : <name>sicudata                                   <runtime-link>static <conditional>@path_options ;
+    searched-lib icudt : : <name>sicudt   <toolset>msvc                     <runtime-link>static <conditional>@path_options ;
+    searched-lib icudt : : <name>sicudt   <toolset>intel <target-os>windows <runtime-link>static <conditional>@path_options ;
+    searched-lib icudt : : <name>this_is_an_invalid_library_name ;
 }
 
 if $(ICU_ICUIN_NAME)
 {
-    lib icuin : : <name>$(ICU_ICUIN_NAME) <conditional>@path_options ;
+    searched-lib icuin : : <name>$(ICU_ICUIN_NAME) <conditional>@path_options ;
     debug-message Using "$(ICU_ICUIN_NAME)" for library "icuin" ;
 }
 else
 {
-    lib icuin : :                                                    <name>icui18n <runtime-link>shared <conditional>@path_options ;
-    lib icuin : : <toolset>msvc                     <variant>debug   <name>icuind  <runtime-link>shared <conditional>@path_options ;
-    lib icuin : : <toolset>msvc                                      <name>icuin   <runtime-link>shared <conditional>@path_options ;
-    lib icuin : : <toolset>intel <target-os>windows <variant>debug   <name>icuind  <runtime-link>shared <conditional>@path_options ;
-    lib icuin : : <toolset>intel <target-os>windows                  <name>icuin   <runtime-link>shared <conditional>@path_options ;
-    lib icuin : :                                                    <name>sicui18n <runtime-link>static <conditional>@path_options ;
-    lib icuin : : <toolset>msvc                     <variant>debug   <name>sicuind  <runtime-link>static <conditional>@path_options ;
-    lib icuin : : <toolset>msvc                                      <name>sicuin   <runtime-link>static <conditional>@path_options ;
-    lib icuin : : <toolset>intel <target-os>windows <variant>debug   <name>sicuind  <runtime-link>static <conditional>@path_options ;
-    lib icuin : : <toolset>intel <target-os>windows                  <name>sicuin   <runtime-link>static <conditional>@path_options ;
-    lib icuin : : <name>this_is_an_invalid_library_name ;
+    searched-lib icuin : :                                                    <name>icui18n <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuin : : <toolset>msvc                     <variant>debug   <name>icuind  <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuin : : <toolset>msvc                                      <name>icuin   <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuin : : <toolset>intel <target-os>windows <variant>debug   <name>icuind  <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuin : : <toolset>intel <target-os>windows                  <name>icuin   <runtime-link>shared <conditional>@path_options ;
+    searched-lib icuin : :                                                    <name>sicui18n <runtime-link>static <conditional>@path_options ;
+    searched-lib icuin : : <toolset>msvc                     <variant>debug   <name>sicuind  <runtime-link>static <conditional>@path_options ;
+    searched-lib icuin : : <toolset>msvc                                      <name>sicuin   <runtime-link>static <conditional>@path_options ;
+    searched-lib icuin : : <toolset>intel <target-os>windows <variant>debug   <name>sicuind  <runtime-link>static <conditional>@path_options ;
+    searched-lib icuin : : <toolset>intel <target-os>windows                  <name>sicuin   <runtime-link>static <conditional>@path_options ;
+    searched-lib icuin : : <name>this_is_an_invalid_library_name ;
 }
 
 ICU_OPTS =
@@ -421,7 +421,7 @@ local cxx_requirements = [ requires
       cxx11_variadic_templates
     ] ;
 
-boost-lib locale
+lib boost_locale
     :
       encoding/codepage.cpp
       shared/date_time.cpp
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 3be48d0..8d791f3 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -6,10 +6,10 @@
 # https://www.boost.org/LICENSE_1_0.txt
 
 import config : requires ;
-import ../../predef/check/predef : check : predef-check ;
+import predef : check : predef-check ;
 
 project : requirements
-    <library>/boost/locale//boost_locale
+    <library>/boost/locale//boost_locale/<link>static
     <warnings>extra
     <toolset>msvc:<warnings-as-errors>on
     <toolset>gcc:<cxxflags>-Wzero-as-null-pointer-constant
