diff --git a/benchmarks/Jamfile b/benchmarks/Jamfile
index ab2f77e8..4aa8ce8b 100644
--- a/benchmarks/Jamfile
+++ b/benchmarks/Jamfile
@@ -4,7 +4,7 @@
 # Boost Software License, Version 1.0. (See accompanying file
 # LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-project boost/ublas/benchmarks
+project
     : requirements <library>/boost/program_options//boost_program_options
     ;
 
diff --git a/benchmarks/opencl/Jamfile b/benchmarks/opencl/Jamfile
index 282d0071..045c9cef 100644
--- a/benchmarks/opencl/Jamfile
+++ b/benchmarks/opencl/Jamfile
@@ -13,7 +13,7 @@ import ../../clblas ;
 using opencl ;
 using clblas ;
 
-project boost/ublas/benchmarks/opencl
+project
     : requirements
       <library>/boost/program_options//boost_program_options
       <toolset>gcc:<cxxflags>-Wno-ignored-attributes
diff --git a/examples/tensor/Jamfile b/examples/tensor/Jamfile
index 101a8a4c..6781de22 100644
--- a/examples/tensor/Jamfile
+++ b/examples/tensor/Jamfile
@@ -6,10 +6,10 @@
 # Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 
-import ../../../../config/checks/config : requires ;
+import config : requires ;
  
 # Project settings
-project boost-ublas-tensor-example
+project
     : requirements
        <define>BOOST_UBLAS_NO_EXCEPTIONS
        <toolset>vacpp:<define>"BOOST_UBLAS_NO_ELEMENT_PROXIES"
@@ -17,9 +17,10 @@ project boost-ublas-tensor-example
        <toolset>clang:<cxxflags>"-Wall -pedantic -Wextra -Wno-unknown-pragmas"
        <toolset>msvc:<cxxflags>"/W4" # == all
        [ requires cxx17_if_constexpr ]
+       <cxxstd>20
     ;
 
-exe access_tensor                      : access_tensor.cpp ;
+exe access_tensor                      : access_tensor.cpp /boost/multiprecision//boost_multiprecision ;
 exe simple_expressions                 : simple_expressions.cpp ;
 exe multiply_tensors_product_function  : multiply_tensors_product_function.cpp ;
 exe multiply_tensors_einstein_notation : multiply_tensors_einstein_notation.cpp ;
diff --git a/test/Jamfile b/test/Jamfile
index f6716e2c..0cbfb9f0 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -47,7 +47,7 @@ UBLAS_TESTSET_SPARSE_COO ?=
 
 
 # Project settings
-project boost-ublas-test
+project
     : requirements
        <define>BOOST_UBLAS_NO_EXCEPTIONS
        <toolset>vacpp:<define>"BOOST_UBLAS_NO_ELEMENT_PROXIES"
@@ -121,6 +121,7 @@ test-suite numeric/uBLAS
       [ run test4.cpp
             test42.cpp
             test43.cpp
+            /boost/math//boost_math
         : : :
             <define>$(UBLAS_TESTSET)
       ]
@@ -165,38 +166,38 @@ test-suite numeric/uBLAS
       ]
       [ run triangular_layout.cpp
       ]
-      [ run comp_mat_erase.cpp
+      [ run comp_mat_erase.cpp /boost/test//boost_test
         : : :
             <toolset>msvc:<asynch-exceptions>on
       ]
-      [ run sparse_view_test.cpp
+      [ run sparse_view_test.cpp /boost/test//boost_test
         : : :
             <toolset>msvc:<asynch-exceptions>on
       ]
-      [ run begin_end.cpp
+      [ run begin_end.cpp /boost/math//boost_math
       ]
-      [ run num_columns.cpp
+      [ run num_columns.cpp /boost/math//boost_math
       ]
-      [ run num_rows.cpp
+      [ run num_rows.cpp /boost/math//boost_math
       ]
-      [ run size.cpp
+      [ run size.cpp /boost/math//boost_math
       ]
-      [ run test_coordinate_matrix_sort.cpp
+      [ run test_coordinate_matrix_sort.cpp /boost/math//boost_math
       ]
-      [ run test_coordinate_matrix_sort.cpp
+      [ run test_coordinate_matrix_sort.cpp /boost/math//boost_math
        :
        :
        : <define>BOOST_UBLAS_COO_ALWAYS_DO_FULL_SORT
        : test_coordinate_matrix_always_do_full_sort
        :
       ]
-      [ run test_complex_norms.cpp
+      [ run test_complex_norms.cpp /boost/math//boost_math
       ]
-      [ run test_scaled_norm.cpp
+      [ run test_scaled_norm.cpp /boost/math//boost_math
        : : :
            <define>BOOST_UBLAS_SCALED_NORM
       ]
-      [ run test_assignment.cpp
+      [ run test_assignment.cpp /boost/timer//boost_timer /boost/math//boost_math
        : : :
            <define>BOOST_UBLAS_COO_ALWAYS_DO_FULL_SORT
       ]
@@ -205,21 +206,21 @@ test-suite numeric/uBLAS
        :
        : <library>/boost/timer//boost_timer
       ]
-      [ run test_ticket7296.cpp
+      [ run test_ticket7296.cpp /boost/math//boost_math
         :
         :
         : <toolset>msvc:<cxxflags>/wd4127 # The program checks that test facilities work fine. The warning appears many times.
         :
         :
       ]
-      [ run test_inplace_solve.cpp
+      [ run test_inplace_solve.cpp /boost/math//boost_math
         : 
         : 
         : <define>$(UBLAS_TESTSET)
         : test_inplace_solve_basic
         :
       ]
-      [ run test_inplace_solve.cpp
+      [ run test_inplace_solve.cpp /boost/math//boost_math
         : 
         : 
         : <define>$(UBLAS_TESTSET_SPARSE)
@@ -227,30 +228,30 @@ test-suite numeric/uBLAS
         : test_inplace_solve_sparse
         :
       ]
-      [ run test_inplace_solve.cpp
+      [ run test_inplace_solve.cpp /boost/math//boost_math
         : 
         : 
         : <define>USE_MAPPED_VECTOR_OF_MAPPED_VECTOR
         : test_inplace_solve_mvov
         :
       ]
-      [ run test_coordinate_vector_inplace_merge.cpp
+      [ run test_coordinate_vector_inplace_merge.cpp /boost/math//boost_math
       ]
-      [ run test_coordinate_matrix_inplace_merge.cpp
+      [ run test_coordinate_matrix_inplace_merge.cpp /boost/math//boost_math
       ]
-      [ run test_banded_storage_layout.cpp
+      [ run test_banded_storage_layout.cpp /boost/math//boost_math
         :
         :
         :
         :
         :
       ]
-      [ run test_fixed_containers.cpp
+      [ run test_fixed_containers.cpp /boost/math//boost_math
         :
         :
         :
       ]
-      [ run test_matrix_vector.cpp
+      [ run test_matrix_vector.cpp /boost/math//boost_math
       ]
       [ compile minimal_allocator_test.cpp ]
     ;
diff --git a/test/opencl/Jamfile b/test/opencl/Jamfile
index cc867922..9da1870e 100644
--- a/test/opencl/Jamfile
+++ b/test/opencl/Jamfile
@@ -13,7 +13,7 @@ import ../../clblas ;
 using opencl ;
 using clblas ;
 
-project boost/ublas/test/opencl
+project
     : requirements
       <toolset>gcc:<cxxflags>-Wno-ignored-attributes
       <toolset>clang:<cxxflags>-Wno-ignored-attributes
diff --git a/test/tensor/Jamfile b/test/tensor/Jamfile
index 908e359e..bcfad63f 100644
--- a/test/tensor/Jamfile
+++ b/test/tensor/Jamfile
@@ -11,7 +11,7 @@
 #  Google and Fraunhofer IOSB, Ettlingen, Germany
 #
 
-import ../../../../config/checks/config : requires ;
+import config : requires ;
 
 project boost/ublas/test/tensor
     : requirements
@@ -22,7 +22,7 @@ project boost/ublas/test/tensor
 
 alias unit_test_framework
     : # sources
-        /boost//unit_test_framework
+        /boost/test//unit_test_framework
     ;
 
 # make aliases explicit so the libraries will only be built when requested
