diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index d0fc589..3b326fb 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -8,6 +8,11 @@
 import testing ;
 import set ;
 
+project : requirements
+    <source>/boost/core//boost_core
+    <source>/boost/lexical_cast//boost_lexical_cast
+    ;
+
 # The special requirement is not ported yet.
 #
 #local rule special-requirements ( toolset variant : properties * )
