diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 079ad44..d15b573 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -8,8 +8,6 @@
 
 # For more information, see http://www.boost.org
 
-project : requirements <implicit-dependency>/boost//headers ;
-
 exe hello_world_slot : hello_world_slot.cpp ;
 
 exe hello_world_multi_slot : hello_world_multi_slot.cpp ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index aacb3a4..c988600 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,8 +13,8 @@
 import testing ;
 
 project
-    : source-location .
     : requirements
+        <source>/boost/test//boost_test
         <hardcode-dll-paths>true
 #        <library>/boost/test//boost_unit_test_framework
 #        <link>static
@@ -23,7 +23,7 @@ project
 rule thread-run ( sources )
 {
     return
-    [ run $(sources) : : : <library>/boost/thread//boost_thread/
+    [ run $(sources) : : : <library>/boost/thread//boost_thread
         <threading>multi ]
     ;
 }
