diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 327cfe9..6ba626a 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine2/example
+project
     : requirements
       <library>/boost/context//boost_context
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-fsplit-stack
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 8cec95c..ca3231a 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -12,11 +12,11 @@ import os ;
 import path ;
 import testing ;
 import toolset ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
-project boost/coroutine2/test
+project
     : requirements
-      <library>../../test/build//boost_unit_test_framework
+      <library>/boost/test//boost_unit_test_framework
       <library>/boost/context//boost_context
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-fsplit-stack
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-DBOOST_USE_SEGMENTED_STACKS
