diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index 04e53d7..0000000
--- a/Jamfile
+++ /dev/null
@@ -1,10 +0,0 @@
-#  Boost.ConceptCheck Library Jamfile
-#
-#  Copyright (c) 2018 James E. King III
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project test ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 4510302..6168d17 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -4,6 +4,10 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 import testing ;
 
+project : requirements
+    <source>/boost/core//libs
+    ;
+
 test-suite concept_check
   : [ link stl_concept_covering.cpp ]
     [ run stl_concept_check.cpp ]
