diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 59139f8..afdcc23 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -4,5 +4,14 @@
 
 import testing ;
 
+project : requirements
+    <source>/boost/core//boost_core
+    <source>/boost/detail//boost_detail
+    <source>/boost/predef//boost_predef
+    <source>/boost/random//boost_random
+    <source>/boost/timer//boost_timer
+    <source>/boost/typeof//boost_typeof
+    ;
+
 run crc_test.cpp ;
 run crc_test2.cpp ;
