diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 4e8f44e..79c9693 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -30,6 +30,7 @@ rule test_all
         <toolset>gcc,<target-os>windows:<linkflags>"-lole32 -loleaut32"
         <host-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
         <host-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+        <library>/boost/interprocess//boost_interprocess
       ] ;
    }
 
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 8193d27..7ab0809 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -25,6 +25,8 @@ rule test_all
       :  # test-files
       :  # requirements
         <host-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+        <library>/boost/core//boost_core
+        <library>/boost/container//boost_container
       ] ;
    }
 
