diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index bcca3d8..8518fe0 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -8,7 +8,6 @@
 import ../../config/checks/config : requires ;
 
 project
-  : source-location .
   : requirements
     [ requires cxx11_rvalue_references ]
     <visibility>hidden
@@ -73,7 +72,7 @@ lib boost_stacktrace_backtrace
     <target-os>linux:<library>dl
     <library>backtrace
     <link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
-    [ check-target-builds ../build//libbacktrace : : <build>no ]
+    [ check-target-builds libbacktrace : : <build>no ]
   : # default build
   : # usage-requirements
     #<link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
@@ -86,7 +85,7 @@ lib boost_stacktrace_addr2line
     <warnings>all
     <target-os>linux:<library>dl
     <link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
-    [ check-target-builds ../build//addr2line : : <build>no ]
+    [ check-target-builds addr2line : : <build>no ]
   : # default build
   : # usage-requirements
     #<link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
@@ -99,7 +98,7 @@ lib boost_stacktrace_basic
     <warnings>all
     <target-os>linux:<library>dl
     <link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
-    [ check-target-builds ../build//WinDbg : <build>no ]
+    [ check-target-builds WinDbg : <build>no ]
   : # default build
   : # usage-requirements
     #<link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
@@ -112,7 +111,7 @@ lib boost_stacktrace_windbg
     <warnings>all
     <library>Dbgeng <library>ole32
     <link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
-    [ check-target-builds ../build//WinDbg : : <build>no ]
+    [ check-target-builds WinDbg : : <build>no ]
   : # default build
   : # usage-requirements
     #<link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
@@ -125,10 +124,8 @@ lib boost_stacktrace_windbg_cached
     <warnings>all
     <library>Dbgeng <library>ole32
     <link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
-    [ check-target-builds ../build//WinDbgCached : : <build>no ]
+    [ check-target-builds WinDbgCached : : <build>no ]
   : # default build
   : # usage-requirements
     #<link>shared:<define>BOOST_STACKTRACE_DYN_LINK=1
   ;
-
-boost-install boost_stacktrace_noop boost_stacktrace_backtrace boost_stacktrace_addr2line boost_stacktrace_basic boost_stacktrace_windbg boost_stacktrace_windbg_cached ;
diff --git a/example/user_config.cpp b/example/user_config.cpp
index a1403cf..3830ccd 100644
--- a/example/user_config.cpp
+++ b/example/user_config.cpp
@@ -4,7 +4,7 @@
 // accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
-#define BOOST_USER_CONFIG <libs/stacktrace/example/user_config.hpp>
+#define BOOST_USER_CONFIG <example/user_config.hpp>
 
 #include <boost/array.hpp>
 #include <exception>    // std::set_terminate, std::abort
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index ffdcdb4..fb8ae3e 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,7 +5,7 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 #
 
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 lib dl : : <link>shared ;
 lib gcc_s ;
@@ -31,6 +31,7 @@ project
     <warnings>all
     <test-info>always_show_run_output
     <visibility>hidden
+    <include>..
   ;
 
 local FORCE_SYMBOL_EXPORT = <target-os>freebsd:<linkflags>"-rdynamic" <target-os>solaris:<linkflags>"-Bdynamic" <target-os>aix:<linkflags>"-rdynamic"
@@ -195,12 +196,12 @@ for local p in [ glob ../example/*.cpp ]
     local additional_dependency = ;
     if $(target_name) = "terminate_handler"
     {
-        additional_dependency = <library>/boost/filesystem//boost_filesystem <library>/boost/system//boost_system <target-os>linux:<library>rt ;
+        additional_dependency = <library>/boost/filesystem//boost_filesystem <library>/boost/system//boost_system <target-os>linux:<library>rt <library>/boost/interprocess//boost_interprocess ;
     }
 
     if $(target_name) = "throwing_st"
     {
-        additional_dependency = [ requires rtti ] ;
+        additional_dependency = [ requires rtti ] <library>/boost/exception//boost_exception ;
     }
 
     stacktrace_tests += [ run $(p) : : : <debug-symbols>on $(LINKSHARED_BT)           $(additional_dependency)    : backtrace_$(p2[1]:B) ] ;
