diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index f1ab8d81..3407fc38 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,7 +7,7 @@
 import path ;
 import regex ;
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 path-constant TOP : . ;
 
@@ -20,6 +20,7 @@ local msvc-flags = /wd4494 ;
 
 project
     : requirements
+        <source>/boost/concept_check//boost_concept_check
         <warnings>pedantic
         <toolset>intel:<warnings>on
 
@@ -145,7 +146,7 @@ run unordered/serialization_tests.cpp
       <toolset>gcc:<optimization>space
       <toolset>clang:<inlining>on
       <toolset>clang:<optimization>space
-      <library>/boost//serialization/<warnings>off ;
+      <library>/boost/serialization//boost_serialization/<warnings>off ;
 
 compile-fail unordered/insert_node_type_fail.cpp : <define>UNORDERED_TEST_MAP      : insert_node_type_fail_map ;
 compile-fail unordered/insert_node_type_fail.cpp : <define>UNORDERED_TEST_MULTIMAP : insert_node_type_fail_multimap ;
@@ -245,7 +246,7 @@ run unordered/serialization_tests.cpp
       <toolset>gcc:<optimization>space
       <toolset>clang:<inlining>on
       <toolset>clang:<optimization>space
-      <library>/boost//serialization/<warnings>off
+      <library>/boost/serialization//boost_serialization/<warnings>off
     : foa_serialization_tests ;
 
 local FOA_EXCEPTION_TESTS =
@@ -348,7 +349,7 @@ run cfoa/serialization_tests.cpp
       <toolset>gcc:<optimization>space
       <toolset>clang:<inlining>on
       <toolset>clang:<optimization>space
-      <library>/boost//serialization/<warnings>off
+      <library>/boost/serialization//boost_serialization/<warnings>off
     : cfoa_serialization_tests ;
 
 alias cfoa_tests :
