diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index c8fe547..0000000
--- a/Jamfile
+++ /dev/null
@@ -1,11 +0,0 @@
-#  Boost.DateTime Library Jamfile
-#
-#  Copyright (c) 2018 James E. King III
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project example ;
-build-project test ;
diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index a8cf24f..872fb74 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -7,10 +7,10 @@
 #
 
 
-project boost/date_time
+project
     : requirements
         <define>DATE_TIME_INLINE
-	<link>shared:<define>BOOST_ALL_DYN_LINK=1
+        <link>shared:<define>BOOST_ALL_DYN_LINK=1
     : usage-requirements
         <define>DATE_TIME_INLINE
         <link>shared:<define>BOOST_DATE_TIME_DYN_LINK=1
@@ -20,5 +20,3 @@ project boost/date_time
 # greg_month.cpp is now just a stub so that there is
 # still a boost_date_time library to link for backward compatibility
 lib boost_date_time : gregorian/greg_month.cpp ;
-
-boost-install boost_date_time ;
diff --git a/example/Jamfile b/example/Jamfile
index 89e0ad1..3be1d36 100644
--- a/example/Jamfile
+++ b/example/Jamfile
@@ -1,4 +1,4 @@
-project libs/date_time/example ;
+project ;
 
 exe dates_as_strings   : gregorian/dates_as_strings.cpp ;
 exe days_alive         : gregorian/days_alive.cpp ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 74da9b9..91fd145 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -43,12 +43,12 @@ run gregorian/testparse_date.cpp ;
 run gregorian/testperiod.cpp ;
 
 run gregorian/testgreg_serialize.cpp
-     ../../serialization/build//boost_serialization
+     /boost/serialization//boost_serialization
      : : : <define>DATE_TIME_XML_SERIALIZE
      : testgreg_serialize_xml ;
 
 run gregorian/testgreg_serialize.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : :
     : testgreg_serialize ;
 
@@ -75,35 +75,35 @@ run posix_time/testtime_formatters.cpp
 
 # text archive tests
 run posix_time/testtime_serialize.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : :
     : testtime_serialize ;
 
 run posix_time/testtime_serialize.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : : <define>BOOST_DATE_TIME_POSIX_TIME_STD_CONFIG
     : testtime_serialize_std_config ;
 
 # xml archive tests
 run posix_time/testtime_serialize.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : :  <define>DATE_TIME_XML_SERIALIZE <define>BOOST_DATE_TIME_POSIX_TIME_STD_CONFIG
     : testtime_serialize_xml_std_config ;
 
 run posix_time/testtime_serialize.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : : <define>DATE_TIME_XML_SERIALIZE
     : testtime_serialize_xml ;
 
 # versioning tests
 run posix_time/testtime_serialize_versioning.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : testtime_serialize_versioning_prev 
     : <define>BOOST_DATE_TIME_POSIX_TIME_STD_CONFIG
     : testtime_serialize_versioning_curr ;
 
 run posix_time/testtime_serialize_versioning.cpp
-    ../../serialization/build//boost_serialization
+    /boost/serialization//boost_serialization
     : : :  <define>BOOST_DATE_TIME_POSIX_TIME_DURATION_VERSION=0
     : testtime_serialize_versioning_prev ;
 
@@ -144,10 +144,9 @@ run local_time/testclocks.cpp ;
 # and basic syntax errors.
 if ! [ os.environ BOOST_DATE_TIME_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS ]
 {
-    local headers_path = [ path.make $(BOOST_ROOT)/libs/date_time/include/boost ] ;
-    for file in [ path.glob-tree $(headers_path) : *.hpp ]
+    for file in [ glob-tree-ex ../include/boost : *.hpp ]
     {
-        local rel_file = [ path.relative-to $(headers_path) $(file) ] ;
+        local rel_file = [ path.relative-to ../include/boost $(file) ] ;
         # Note: The test name starts with '~' in order to group these tests in the test report table, preferably at the end.
         #       All '/' are replaced with '-' because apparently test scripts have a problem with test names containing slashes.
         local test_name = [ regex.replace ~hdr/$(rel_file) "/" "-" ] ;
