diff --git a/tool/build/Jamfile.v2 b/tool/build/Jamfile.v2
index dff6030..a41afb9 100644
--- a/tool/build/Jamfile.v2
+++ b/tool/build/Jamfile.v2
@@ -34,12 +34,12 @@ project
 exe wave
     :
     ../cpp.cpp 
-    /boost//wave
-    /boost//program_options
-    /boost//filesystem
-    /boost//serialization
-    /boost//system
-    /boost//thread
+    /boost/wave//boost_wave
+    /boost/program_options//boost_program_options
+    /boost/filesystem//boost_filesystem
+    /boost/serialization//boost_serialization
+    /boost/system//boost_system
+    /boost/thread//boost_thread
     /boost/timer//boost_timer/<link>static
     :
     <threading>multi
