diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 3789759..03af444 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,11 +7,15 @@
 
 # For more information, see http://www.boost.org
 
+require-b2 5.0.1 ;
+
+import-search /boost/config/checks ;
+
 # bring in rules for testing
 import testing ;
 import path ;
 
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 local RDYNAMIC = <target-os>freebsd:<linkflags>"-rdynamic" <target-os>solaris:<linkflags>"-Bdynamic" <target-os>aix:<linkflags>"-rdynamic"
     <target-os>qnxnto,<toolset>qcc:<linkflags>"-Bdynamic" <target-os>qnxnto,<toolset>gcc:<linkflags>"-rdynamic"
