diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index e043da0..cce239f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,7 +5,9 @@
 
 project
     : requirements
-		<include>../include
+        <source>/boost/core//libs
+        <source>/boost/exception//libs
+        <source>/boost/test//libs
     ;
 
 variant debug_qvm_hpp : debug : <define>BOOST_QVM_TEST_SINGLE_HEADER="\\\"qvm.hpp\\\"" ;
