diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index fdaef72..9ae09ed 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,7 +13,7 @@ run base_from_member_ref_test.cpp ;
 
 run binary_test.cpp ;
 
-run call_traits_test.cpp : -u ;
+run call_traits_test.cpp : -u : : <source>/boost/type_traits//testing ;
 
 run compressed_pair_test.cpp ;
 run compressed_pair_final_test.cpp ;
@@ -31,7 +31,7 @@ run string_ref_test2.cpp ;
 run string_ref_test_io.cpp ;
 # compile-fail string_view_from_rvalue.cpp ;
 compile string_view_constexpr_test1.cpp ;
-run string_view_test1.cpp ;
+run string_view_test1.cpp : : : <source>/boost/container_hash//boost_container_hash ;
 run string_view_test2.cpp ;
 run string_view_test_io.cpp ;
 
diff --git a/test/call_traits_test.cpp b/test/call_traits_test.cpp
index 57ad67c..ea9d533 100644
--- a/test/call_traits_test.cpp
+++ b/test/call_traits_test.cpp
@@ -18,8 +18,9 @@
 #include <typeinfo>
 #include <boost/call_traits.hpp>
 
-#include <libs/type_traits/test/test.hpp>
-#include <libs/type_traits/test/check_type.hpp>
+// type_traits/test utilities
+#include <test.hpp>
+#include <check_type.hpp>
 
 #ifdef BOOST_MSVC
 #pragma warning(disable:4181) // : warning C4181: qualifier applied to reference type; ignored
