diff --git a/examples/user_defined_typeinfo.cpp b/examples/user_defined_typeinfo.cpp
index fe73b7d..889b0ae 100644
--- a/examples/user_defined_typeinfo.cpp
+++ b/examples/user_defined_typeinfo.cpp
@@ -14,7 +14,7 @@
 */
 
 // BOOST_TYPE_INDEX_USER_TYPEINDEX must be defined *BEFORE* first inclusion of <boost/type_index.hpp>
-#define BOOST_TYPE_INDEX_USER_TYPEINDEX <boost/../libs/type_index/examples/user_defined_typeinfo.hpp>
+#define BOOST_TYPE_INDEX_USER_TYPEINDEX <user_defined_typeinfo.hpp>
 #include <boost/type_index.hpp>
 //] [/type_index_my_type_index_worldwide_macro]
 
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index adc8cb2..dc9f2e1 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -47,10 +47,10 @@ exe testing_crossmodule_anonymous_no_rtti : testing_crossmodule_anonymous.cpp te
 
 test-suite type_index
   : 
-    [ run type_index_test.cpp ]
-    [ run type_index_runtime_cast_test.cpp ]
+    [ run type_index_test.cpp /boost/lexical_cast//boost_lexical_cast ]
+    [ run type_index_runtime_cast_test.cpp /boost/smart_ptr//boost_smart_ptr ]
     [ run type_index_constexpr_test.cpp ]
-    [ run type_index_test.cpp : : : <rtti>off $(norttidefines) : type_index_test_no_rtti ]
+    [ run type_index_test.cpp /boost/lexical_cast//boost_lexical_cast : : : <rtti>off $(norttidefines) : type_index_test_no_rtti ]
     [ run ctti_print_name.cpp : : : <test-info>always_show_run_output ]
     [ run testing_crossmodule.cpp test_lib_rtti ]
     [ run testing_crossmodule.cpp test_lib_nortti : : : <rtti>off $(norttidefines) : testing_crossmodule_no_rtti ]
@@ -77,13 +77,13 @@ test-suite type_index
 for local p in [ glob ../examples/*.cpp ]
 {
     # RTTI on
-    type_index += [ run $(p) ] ;
+    type_index += [ run $(p) /boost/unordered//boost_unordered : : : <include>../examples ] ;
 
     # RTTI off
     local target_name = $(p[1]:B)_no_rtti ;
     if $(target_name) != "table_of_names_no_rtti"
     {
-        type_index += [ run $(p) : : : <rtti>off $(norttidefines) : $(target_name) ] ;
+        type_index += [ run $(p) /boost/unordered//boost_unordered : : : <include>../examples <rtti>off $(norttidefines) : $(target_name) ] ;
     }
 }
 
