diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index c55393e..0000000
--- a/Jamfile
+++ /dev/null
@@ -1,11 +0,0 @@
-#  Boost.Container Library Jamfile
-#
-#  Copyright (c) 2018 Ion Gaztanaga
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project example ;
-build-project test ;
diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 3adb451..c6b2677 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -6,7 +6,7 @@
 
 import sequence ;
 
-project boost/container
+project
     : source-location ../src
     : usage-requirements  # pass these requirement to dependents (i.e. users)
       <link>shared:<define>BOOST_CONTAINER_DYN_LINK=1
@@ -19,4 +19,4 @@ lib boost_container
      <link>static:<define>BOOST_CONTAINER_STATIC_LINK=1
    ;
 
-boost-install boost_container ;
\ No newline at end of file
+# boost-install boost_container ;
\ No newline at end of file
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 2b4995b..5acb3c7 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,6 +13,11 @@
 
 project
     : requirements 
+      <source>/boost/algorithm//boost_algorithm
+      <source>/boost/iterator//boost_iterator
+      <source>/boost/utility//boost_utility
+      <source>/boost/tuple//boost_tuple
+
       <link>shared:<define>BOOST_CONTAINER_DYN_LINK=1 
       <toolset>gcc-cygwin:<link>static
     ;
