diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 3adb451..bdfbeb5 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -6,7 +6,7 @@
 
 import sequence ;
 
-project boost/container
+project
     : source-location ../src
     : usage-requirements  # pass these requirement to dependents (i.e. users)
       <link>shared:<define>BOOST_CONTAINER_DYN_LINK=1
@@ -18,5 +18,3 @@ lib boost_container
    : <link>shared:<define>BOOST_CONTAINER_DYN_LINK=1
      <link>static:<define>BOOST_CONTAINER_STATIC_LINK=1
    ;
-
-boost-install boost_container ;
\ No newline at end of file
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 2b4995b..5acb3c7 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,6 +13,11 @@
 
 project
     : requirements 
+      <source>/boost/algorithm//boost_algorithm
+      <source>/boost/iterator//boost_iterator
+      <source>/boost/utility//boost_utility
+      <source>/boost/tuple//boost_tuple
+
       <link>shared:<define>BOOST_CONTAINER_DYN_LINK=1 
       <toolset>gcc-cygwin:<link>static
     ;
