diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 5e48af9..a89da91 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -7,72 +7,54 @@ exe interval
     :
         interval_/interval.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe interval_container
     :
         interval_container_/interval_container.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe overlap_counter
     :
         overlap_counter_/overlap_counter.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe party
     :
         party_/party.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe std_copy
     :
         std_copy_/std_copy.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe std_transform
     :
         std_transform_/std_transform.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe custom_interval
     :
         custom_interval_/custom_interval.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe dynamic_interval
     :
         dynamic_interval_/dynamic_interval.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe static_interval
     :
         static_interval_/static_interval.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 # Examples using boost_date_time
@@ -80,48 +62,36 @@ exe boost_party
     :
         boost_party_/boost_party.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe partys_height_average
     :
         partys_height_average_/partys_height_average.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe partys_tallest_guests
     :
         partys_tallest_guests_/partys_tallest_guests.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe man_power
     :
         man_power_/man_power.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe month_and_week_grid
     :
         month_and_week_grid_/month_and_week_grid.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe user_groups
     :
         user_groups_/user_groups.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 # Projects
@@ -129,8 +99,6 @@ exe large_bitset
     :
         large_bitset_/large_bitset.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 
@@ -139,15 +107,11 @@ exe itvset_shell
     :
         itvset_shell_/itvset_shell.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
 exe splititvmap_shell
     :
         splititvmap_shell_/splititvmap_shell.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
 
diff --git a/example/boost_party_/Jamfile.v2 b/example/boost_party_/Jamfile.v2
index b9a856b..dab1f9a 100644
--- a/example/boost_party_/Jamfile.v2
+++ b/example/boost_party_/Jamfile.v2
@@ -6,6 +6,4 @@ exe boost_party
     :
         boost_party.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
diff --git a/example/custom_interval_/Jamfile.v2 b/example/custom_interval_/Jamfile.v2
index 404abbc..8e9dc31 100644
--- a/example/custom_interval_/Jamfile.v2
+++ b/example/custom_interval_/Jamfile.v2
@@ -6,6 +6,4 @@ exe custom_interval
     :
         custom_interval.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
diff --git a/example/partys_height_average_/Jamfile.v2 b/example/partys_height_average_/Jamfile.v2
index cc035f2..2f17c26 100644
--- a/example/partys_height_average_/Jamfile.v2
+++ b/example/partys_height_average_/Jamfile.v2
@@ -6,6 +6,4 @@ exe partys_height_average
     :
         partys_height_average.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
diff --git a/example/partys_tallest_guests_/Jamfile.v2 b/example/partys_tallest_guests_/Jamfile.v2
index bf2ad30..779dd76 100644
--- a/example/partys_tallest_guests_/Jamfile.v2
+++ b/example/partys_tallest_guests_/Jamfile.v2
@@ -6,6 +6,4 @@ exe partys_tallest_guests
     :
         partys_tallest_guests.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
     ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 3cd9583..0c0c43b 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -11,7 +11,7 @@ project
       <library>/boost/test//boost_unit_test_framework
       <library>/boost/chrono//boost_chrono
       <link>static
-      <include>../../..
+      <include>.
     ;
 
 test-suite "itl"
diff --git a/test/cmp_clang_ttp_passing_/cmp_clang_ttp_passing.cpp b/test/cmp_clang_ttp_passing_/cmp_clang_ttp_passing.cpp
index 9d723eb..91e7860 100644
--- a/test/cmp_clang_ttp_passing_/cmp_clang_ttp_passing.cpp
+++ b/test/cmp_clang_ttp_passing_/cmp_clang_ttp_passing.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2011-2011: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::cmp_clang_ttp_passing unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <boost/config.hpp>
 #include "../unit_test_unwarned.hpp"
 
diff --git a/test/ex_boost_party_/ex_boost_party.cpp b/test/ex_boost_party_/ex_boost_party.cpp
index a50a8f6..bfc2731 100644
--- a/test/ex_boost_party_/ex_boost_party.cpp
+++ b/test/ex_boost_party_/ex_boost_party.cpp
@@ -7,7 +7,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::example_boost_party unit test
 
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include "../unit_test_unwarned.hpp"
 //#include <boost/icl/set.hpp> // Needed for implicit calls of operator << on
 //JODO CLANG                   // GuestSets via test macros.
diff --git a/test/fast_stat_interval_map_/fast_stat_interval_map.cpp b/test/fast_stat_interval_map_/fast_stat_interval_map.cpp
index d9fac95..80f2429 100644
--- a/test/fast_stat_interval_map_/fast_stat_interval_map.cpp
+++ b/test/fast_stat_interval_map_/fast_stat_interval_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2011: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_icl_interval_/fastest_icl_interval.cpp b/test/fastest_icl_interval_/fastest_icl_interval.cpp
index 8a6d799..1327eb2 100644
--- a/test/fastest_icl_interval_/fastest_icl_interval.cpp
+++ b/test/fastest_icl_interval_/fastest_icl_interval.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_icl_map_/fastest_icl_map.cpp b/test/fastest_icl_map_/fastest_icl_map.cpp
index ecd6bc4..8c8c1c4 100644
--- a/test/fastest_icl_map_/fastest_icl_map.cpp
+++ b/test/fastest_icl_map_/fastest_icl_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::icl_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_map_/fastest_interval_map.cpp b/test/fastest_interval_map_/fastest_interval_map.cpp
index b1eb616..f0a9866 100644
--- a/test/fastest_interval_map_/fastest_interval_map.cpp
+++ b/test/fastest_interval_map_/fastest_interval_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_map_infix_/fastest_interval_map_infix.cpp b/test/fastest_interval_map_infix_/fastest_interval_map_infix.cpp
index d792065..9ea29f1 100644
--- a/test/fastest_interval_map_infix_/fastest_interval_map_infix.cpp
+++ b/test/fastest_interval_map_infix_/fastest_interval_map_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_map_infix_mixed_/fastest_interval_map_infix_mixed.cpp b/test/fastest_interval_map_infix_mixed_/fastest_interval_map_infix_mixed.cpp
index a4cbcd9..dfe1c0b 100644
--- a/test/fastest_interval_map_infix_mixed_/fastest_interval_map_infix_mixed.cpp
+++ b/test/fastest_interval_map_infix_mixed_/fastest_interval_map_infix_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_infix_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_map_mixed2_/fastest_interval_map_mixed2.cpp b/test/fastest_interval_map_mixed2_/fastest_interval_map_mixed2.cpp
index 072c105..0c8b7e6 100644
--- a/test/fastest_interval_map_mixed2_/fastest_interval_map_mixed2.cpp
+++ b/test/fastest_interval_map_mixed2_/fastest_interval_map_mixed2.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_map_mixed_/fastest_interval_map_mixed.cpp b/test/fastest_interval_map_mixed_/fastest_interval_map_mixed.cpp
index fd8972f..5885fc6 100644
--- a/test/fastest_interval_map_mixed_/fastest_interval_map_mixed.cpp
+++ b/test/fastest_interval_map_mixed_/fastest_interval_map_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_set_/fastest_interval_set.cpp b/test/fastest_interval_set_/fastest_interval_set.cpp
index 874ca86..566e2f1 100644
--- a/test/fastest_interval_set_/fastest_interval_set.cpp
+++ b/test/fastest_interval_set_/fastest_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_set_infix_/fastest_interval_set_infix.cpp b/test/fastest_interval_set_infix_/fastest_interval_set_infix.cpp
index ede5b5f..4023ff4 100644
--- a/test/fastest_interval_set_infix_/fastest_interval_set_infix.cpp
+++ b/test/fastest_interval_set_infix_/fastest_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_interval_set_mixed_/fastest_interval_set_mixed.cpp b/test/fastest_interval_set_mixed_/fastest_interval_set_mixed.cpp
index 455bcc7..dd0cc86 100644
--- a/test/fastest_interval_set_mixed_/fastest_interval_set_mixed.cpp
+++ b/test/fastest_interval_set_mixed_/fastest_interval_set_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_partial_icl_quantifier_/fastest_partial_icl_quantifier.cpp b/test/fastest_partial_icl_quantifier_/fastest_partial_icl_quantifier.cpp
index 8ead79f..a40b5a6 100644
--- a/test/fastest_partial_icl_quantifier_/fastest_partial_icl_quantifier.cpp
+++ b/test/fastest_partial_icl_quantifier_/fastest_partial_icl_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::partial_icl_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_partial_interval_quantifier_/fastest_partial_interval_quantifier.cpp b/test/fastest_partial_interval_quantifier_/fastest_partial_interval_quantifier.cpp
index 51570be..7778828 100644
--- a/test/fastest_partial_interval_quantifier_/fastest_partial_interval_quantifier.cpp
+++ b/test/fastest_partial_interval_quantifier_/fastest_partial_interval_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::partial_interval_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_separate_interval_set_/fastest_separate_interval_set.cpp b/test/fastest_separate_interval_set_/fastest_separate_interval_set.cpp
index a53cafd..768b92c 100644
--- a/test/fastest_separate_interval_set_/fastest_separate_interval_set.cpp
+++ b/test/fastest_separate_interval_set_/fastest_separate_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::separate_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_separate_interval_set_infix_/fastest_separate_interval_set_infix.cpp b/test/fastest_separate_interval_set_infix_/fastest_separate_interval_set_infix.cpp
index 247b134..b13d66e 100644
--- a/test/fastest_separate_interval_set_infix_/fastest_separate_interval_set_infix.cpp
+++ b/test/fastest_separate_interval_set_infix_/fastest_separate_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::separate_interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_set_icl_set_/fastest_set_icl_set.cpp b/test/fastest_set_icl_set_/fastest_set_icl_set.cpp
index d40d3e7..8a6ecad 100644
--- a/test/fastest_set_icl_set_/fastest_set_icl_set.cpp
+++ b/test/fastest_set_icl_set_/fastest_set_icl_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::set_itl_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_set_interval_set_/fastest_set_interval_set.cpp b/test/fastest_set_interval_set_/fastest_set_interval_set.cpp
index 6f6676e..7753cca 100644
--- a/test/fastest_set_interval_set_/fastest_set_interval_set.cpp
+++ b/test/fastest_set_interval_set_/fastest_set_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::set_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_split_interval_map_/fastest_split_interval_map.cpp b/test/fastest_split_interval_map_/fastest_split_interval_map.cpp
index 35c335f..84cdb54 100644
--- a/test/fastest_split_interval_map_/fastest_split_interval_map.cpp
+++ b/test/fastest_split_interval_map_/fastest_split_interval_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_split_interval_map_infix_/fastest_split_interval_map_infix.cpp b/test/fastest_split_interval_map_infix_/fastest_split_interval_map_infix.cpp
index 8f4bd5b..8ace89e 100644
--- a/test/fastest_split_interval_map_infix_/fastest_split_interval_map_infix.cpp
+++ b/test/fastest_split_interval_map_infix_/fastest_split_interval_map_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_map_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_split_interval_set_/fastest_split_interval_set.cpp b/test/fastest_split_interval_set_/fastest_split_interval_set.cpp
index 1d034de..ef6e9d0 100644
--- a/test/fastest_split_interval_set_/fastest_split_interval_set.cpp
+++ b/test/fastest_split_interval_set_/fastest_split_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_split_interval_set_infix_/fastest_split_interval_set_infix.cpp b/test/fastest_split_interval_set_infix_/fastest_split_interval_set_infix.cpp
index 179066b..86253dc 100644
--- a/test/fastest_split_interval_set_infix_/fastest_split_interval_set_infix.cpp
+++ b/test/fastest_split_interval_set_infix_/fastest_split_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_total_icl_quantifier_/fastest_total_icl_quantifier.cpp b/test/fastest_total_icl_quantifier_/fastest_total_icl_quantifier.cpp
index 48ba0fd..da6b8d4 100644
--- a/test/fastest_total_icl_quantifier_/fastest_total_icl_quantifier.cpp
+++ b/test/fastest_total_icl_quantifier_/fastest_total_icl_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::total_icl_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fastest_total_interval_quantifier_/fastest_total_interval_quantifier.cpp b/test/fastest_total_interval_quantifier_/fastest_total_interval_quantifier.cpp
index 749d5bf..ccc405d 100644
--- a/test/fastest_total_interval_quantifier_/fastest_total_interval_quantifier.cpp
+++ b/test/fastest_total_interval_quantifier_/fastest_total_interval_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::total_interval_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/fix_icl_after_thread_/fix_icl_after_thread.cpp b/test/fix_icl_after_thread_/fix_icl_after_thread.cpp
index ea4cfa7..3eb8390 100644
--- a/test/fix_icl_after_thread_/fix_icl_after_thread.cpp
+++ b/test/fix_icl_after_thread_/fix_icl_after_thread.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2011-2011: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::fix_icl_after_thread unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include "../unit_test_unwarned.hpp"
 
 // #include <boost/thread.hpp> MEMO: The problem occured when using thread.hpp
diff --git a/test/fix_tickets_/fix_tickets.cpp b/test/fix_tickets_/fix_tickets.cpp
index 0232c31..bc2198a 100644
--- a/test/fix_tickets_/fix_tickets.cpp
+++ b/test/fix_tickets_/fix_tickets.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2011-2011: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::fix_icl_after_thread unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include "../unit_test_unwarned.hpp"
 
 //#define BOOST_ICL_USE_STATIC_BOUNDED_INTERVALS
diff --git a/test/test_casual_/test_casual.cpp b/test/test_casual_/test_casual.cpp
index 584954b..be750b6 100644
--- a/test/test_casual_/test_casual.cpp
+++ b/test/test_casual_/test_casual.cpp
@@ -9,7 +9,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
 
 #define BOOST_ICL_TEST_CHRONO
 
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 
 #include <string>
 #include <vector>
diff --git a/test/test_changing_interval_defaults_/test_changing_interval_defaults.cpp b/test/test_changing_interval_defaults_/test_changing_interval_defaults.cpp
index 9501436..21f7c93 100644
--- a/test/test_changing_interval_defaults_/test_changing_interval_defaults.cpp
+++ b/test/test_changing_interval_defaults_/test_changing_interval_defaults.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::test_doc_code unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 
 #include <limits>
 #include <complex>
diff --git a/test/test_combinable_/test_combinable.cpp b/test/test_combinable_/test_combinable.cpp
index 7860795..132aed3 100644
--- a/test/test_combinable_/test_combinable.cpp
+++ b/test/test_combinable_/test_combinable.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::casual unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_doc_code_/test_doc_code.cpp b/test/test_doc_code_/test_doc_code.cpp
index 7339a5f..2b2972f 100644
--- a/test/test_doc_code_/test_doc_code.cpp
+++ b/test/test_doc_code_/test_doc_code.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::test_doc_code unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 
 #include <limits>
 #include <complex>
diff --git a/test/test_icl_interval_/test_icl_interval.cpp b/test/test_icl_interval_/test_icl_interval.cpp
index fb9db03..7380cf8 100644
--- a/test/test_icl_interval_/test_icl_interval.cpp
+++ b/test/test_icl_interval_/test_icl_interval.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_icl_map_/test_icl_map.cpp b/test/test_icl_map_/test_icl_map.cpp
index f433459..6e4af95 100644
--- a/test/test_icl_map_/test_icl_map.cpp
+++ b/test/test_icl_map_/test_icl_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::icl_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_map_/test_interval_map.cpp b/test/test_interval_map_/test_interval_map.cpp
index 2b41e8e..ef7866b 100644
--- a/test/test_interval_map_/test_interval_map.cpp
+++ b/test/test_interval_map_/test_interval_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_map_infix_/test_interval_map_infix.cpp b/test/test_interval_map_infix_/test_interval_map_infix.cpp
index b590535..fae62b0 100644
--- a/test/test_interval_map_infix_/test_interval_map_infix.cpp
+++ b/test/test_interval_map_infix_/test_interval_map_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_map_infix_mixed_/test_interval_map_infix_mixed.cpp b/test/test_interval_map_infix_mixed_/test_interval_map_infix_mixed.cpp
index 3390c16..c988033 100644
--- a/test/test_interval_map_infix_mixed_/test_interval_map_infix_mixed.cpp
+++ b/test/test_interval_map_infix_mixed_/test_interval_map_infix_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_infix_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_map_mixed2_/test_interval_map_mixed2.cpp b/test/test_interval_map_mixed2_/test_interval_map_mixed2.cpp
index 9b89542..e50b4b5 100644
--- a/test/test_interval_map_mixed2_/test_interval_map_mixed2.cpp
+++ b/test/test_interval_map_mixed2_/test_interval_map_mixed2.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_map_mixed_/test_interval_map_mixed.cpp b/test/test_interval_map_mixed_/test_interval_map_mixed.cpp
index 55bac53..d689b13 100644
--- a/test/test_interval_map_mixed_/test_interval_map_mixed.cpp
+++ b/test/test_interval_map_mixed_/test_interval_map_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_map_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_set_/test_interval_set.cpp b/test/test_interval_set_/test_interval_set.cpp
index c2ee2ea..c1ec330 100644
--- a/test/test_interval_set_/test_interval_set.cpp
+++ b/test/test_interval_set_/test_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <vector>
 #include <boost/mpl/list.hpp>
diff --git a/test/test_interval_set_infix_/test_interval_set_infix.cpp b/test/test_interval_set_infix_/test_interval_set_infix.cpp
index 8ef0eca..ad0170a 100644
--- a/test/test_interval_set_infix_/test_interval_set_infix.cpp
+++ b/test/test_interval_set_infix_/test_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_interval_set_mixed_/test_interval_set_mixed.cpp b/test/test_interval_set_mixed_/test_interval_set_mixed.cpp
index 04a7715..bdb88b4 100644
--- a/test/test_interval_set_mixed_/test_interval_set_mixed.cpp
+++ b/test/test_interval_set_mixed_/test_interval_set_mixed.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::interval_set_mixed unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_misc_/test_misc.cpp b/test/test_misc_/test_misc.cpp
index 8d3961a..c48865f 100644
--- a/test/test_misc_/test_misc.cpp
+++ b/test/test_misc_/test_misc.cpp
@@ -9,7 +9,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
 
 #define BOOST_ICL_TEST_CHRONO
 
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <vector>
 #include <boost/mpl/list.hpp>
diff --git a/test/test_partial_icl_quantifier_/test_partial_icl_quantifier.cpp b/test/test_partial_icl_quantifier_/test_partial_icl_quantifier.cpp
index 68d022b..64d00a1 100644
--- a/test/test_partial_icl_quantifier_/test_partial_icl_quantifier.cpp
+++ b/test/test_partial_icl_quantifier_/test_partial_icl_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::partial_icl_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_partial_interval_quantifier_/test_partial_interval_quantifier.cpp b/test/test_partial_interval_quantifier_/test_partial_interval_quantifier.cpp
index 7288864..435703a 100644
--- a/test/test_partial_interval_quantifier_/test_partial_interval_quantifier.cpp
+++ b/test/test_partial_interval_quantifier_/test_partial_interval_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::partial_interval_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_separate_interval_set_/test_separate_interval_set.cpp b/test/test_separate_interval_set_/test_separate_interval_set.cpp
index 0d868d1..e0fa9f6 100644
--- a/test/test_separate_interval_set_/test_separate_interval_set.cpp
+++ b/test/test_separate_interval_set_/test_separate_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::separate_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_separate_interval_set_infix_/test_separate_interval_set_infix.cpp b/test/test_separate_interval_set_infix_/test_separate_interval_set_infix.cpp
index 445b575..e857173 100644
--- a/test/test_separate_interval_set_infix_/test_separate_interval_set_infix.cpp
+++ b/test/test_separate_interval_set_infix_/test_separate_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::separate_interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_set_icl_set_/test_set_icl_set.cpp b/test/test_set_icl_set_/test_set_icl_set.cpp
index 6610286..8812950 100644
--- a/test/test_set_icl_set_/test_set_icl_set.cpp
+++ b/test/test_set_icl_set_/test_set_icl_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::set_itl_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_set_interval_set_/test_set_interval_set.cpp b/test/test_set_interval_set_/test_set_interval_set.cpp
index abb05ef..2a14de8 100644
--- a/test/test_set_interval_set_/test_set_interval_set.cpp
+++ b/test/test_set_interval_set_/test_set_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::set_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_split_interval_map_/test_split_interval_map.cpp b/test/test_split_interval_map_/test_split_interval_map.cpp
index 74b49d1..2cc04e7 100644
--- a/test/test_split_interval_map_/test_split_interval_map.cpp
+++ b/test/test_split_interval_map_/test_split_interval_map.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_map unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_split_interval_map_infix_/test_split_interval_map_infix.cpp b/test/test_split_interval_map_infix_/test_split_interval_map_infix.cpp
index 13ac486..4e9a225 100644
--- a/test/test_split_interval_map_infix_/test_split_interval_map_infix.cpp
+++ b/test/test_split_interval_map_infix_/test_split_interval_map_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_map_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_split_interval_set_/test_split_interval_set.cpp b/test/test_split_interval_set_/test_split_interval_set.cpp
index d1e20ce..fea257a 100644
--- a/test/test_split_interval_set_/test_split_interval_set.cpp
+++ b/test/test_split_interval_set_/test_split_interval_set.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_set unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_split_interval_set_infix_/test_split_interval_set_infix.cpp b/test/test_split_interval_set_infix_/test_split_interval_set_infix.cpp
index fe73987..90550f2 100644
--- a/test/test_split_interval_set_infix_/test_split_interval_set_infix.cpp
+++ b/test/test_split_interval_set_infix_/test_split_interval_set_infix.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::split_interval_set_infix unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_total_icl_quantifier_/test_total_icl_quantifier.cpp b/test/test_total_icl_quantifier_/test_total_icl_quantifier.cpp
index 95d2758..ea00677 100644
--- a/test/test_total_icl_quantifier_/test_total_icl_quantifier.cpp
+++ b/test/test_total_icl_quantifier_/test_total_icl_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::total_icl_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_total_interval_quantifier_/test_total_interval_quantifier.cpp b/test/test_total_interval_quantifier_/test_total_interval_quantifier.cpp
index 3312785..a98c04c 100644
--- a/test/test_total_interval_quantifier_/test_total_interval_quantifier.cpp
+++ b/test/test_total_interval_quantifier_/test_total_interval_quantifier.cpp
@@ -6,7 +6,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
            http://www.boost.org/LICENSE_1_0.txt)
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::total_interval_quantifier unit test
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <string>
 #include <boost/mpl/list.hpp>
 #include "../unit_test_unwarned.hpp"
diff --git a/test/test_type_lists.hpp b/test/test_type_lists.hpp
index 78e6df3..46c6ad4 100644
--- a/test/test_type_lists.hpp
+++ b/test/test_type_lists.hpp
@@ -23,7 +23,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
 
 #ifdef BOOST_ICL_TEST_CHRONO
 #   define BOOST_CHRONO_EXTENSIONS
-#   include <libs/icl/test/chrono/utility.hpp>
+#   include <chrono/utility.hpp>
 
     namespace boch  = boost::chrono;
 #endif
diff --git a/test/test_type_traits_/test_type_traits.cpp b/test/test_type_traits_/test_type_traits.cpp
index b24c6d2..5abe196 100644
--- a/test/test_type_traits_/test_type_traits.cpp
+++ b/test/test_type_traits_/test_type_traits.cpp
@@ -7,7 +7,7 @@ Copyright (c) 2008-2009: Joachim Faulhaber
 +-----------------------------------------------------------------------------*/
 #define BOOST_TEST_MODULE icl::test_type_traits unit test
 
-#include <libs/icl/test/disable_test_warnings.hpp>
+#include <disable_test_warnings.hpp>
 #include <limits>
 #include <complex>
 #include <string>
