diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index a4dc9c3..41914be 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -24,6 +24,7 @@ exe complex_structs
 
 exe composite_keys
     : composite_keys.cpp
+      /boost/tokenizer//boost_tokenizer
     : <include>$(BOOST_ROOT)
     ;
 
@@ -34,11 +35,13 @@ exe fun_key
 
 exe hashed
     : hashed.cpp
+      /boost/tokenizer//boost_tokenizer
     : <include>$(BOOST_ROOT)
     ;
 
 exe ip_allocator
     : ip_allocator.cpp
+      /boost/interprocess//boost_interprocess
     : <include>$(BOOST_ROOT) <threading>multi
     ;
 
@@ -49,16 +52,19 @@ exe non_default_ctor
 
 exe random_access
     : random_access.cpp
+      /boost/tokenizer//boost_tokenizer
     : <include>$(BOOST_ROOT)
     ;
 
 exe rearrange
     : rearrange.cpp
+      /boost/random//boost_random
     : <include>$(BOOST_ROOT)
     ;
 
 exe sequenced
     : sequenced.cpp
+      /boost/tokenizer//boost_tokenizer
     : <include>$(BOOST_ROOT)
     ;
 
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index fea3033..01aded7 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -46,7 +46,8 @@ test-suite "multi_index" :
     [ run test_conv_iterators.cpp   test_conv_iterators_main.cpp   ]
     [ run test_copy_assignment.cpp  test_copy_assignment_main.cpp  ]
     [ run test_hash_ops.cpp         test_hash_ops_main.cpp         ]
-    [ run test_iterators.cpp        test_iterators_main.cpp        ]
+    [ run test_iterators.cpp        test_iterators_main.cpp
+        /boost/foreach//boost_foreach                              ]
     [ run test_key.cpp              test_key_main.cpp
         : : :
         [ check-target-builds boost_multi_index_key_supported
@@ -71,6 +72,6 @@ test-suite "multi_index" :
     [ run test_special_set_ops.cpp  test_special_set_ops_main.cpp  ]
     [ run test_update.cpp           test_update_main.cpp 
         : : : 
-        -<tag>@$(BOOST_JAMROOT_MODULE)%$(BOOST_JAMROOT_MODULE).tag
+        -<tag>@%boostcpp.tag
         <tag>@change-test_update-exe-name                          ]
     ;
