diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 16787e3..bb84d70 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -8,6 +8,18 @@
 
 import testing ;
 
+project 
+    : requirements
+        <source>/boost/core//boost_core
+        <source>/boost/random//boost_random
+
+        <warnings>all
+        <toolset>intel:<warnings>on
+        <toolset>intel-win:<cxxflags>"/bigobj"
+        <toolset>gcc:<cxxflags>"-pedantic -Wall -Wstrict-aliasing -fstrict-aliasing -Wno-long-long"
+        <toolset>msvc:<cxxflags>"/W4 /bigobj"
+    ;
+
 run polygon_point_test.cpp ;
 run polygon_segment_test.cpp ;
 run polygon_interval_test.cpp ;
