diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 848400c..a7772d7 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -24,7 +24,7 @@ run example10.cpp ;
 run example11.cpp ;
 run example13.cpp ;
 run example14.cpp ;
-run example15.cpp ;
+run example15.cpp : : : <source>/boost/rational//boost_rational ;
 run example16.cpp ;
 compile-fail example17.cpp ;
 run example18.cpp ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index f846a85..18d972f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,7 +7,7 @@
 #  http://www.boost.org/LICENSE_1_0.txt
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 project 
     : requirements 
         [ requires cxx14_constexpr ]
@@ -65,7 +65,7 @@ run test_performance.cpp # sources
     :  <variant>debug:<build>no # requirements
     ;
 run test_range.cpp ;
-run test_rational.cpp ;
+run test_rational.cpp : : : <source>/boost/rational//boost_rational ;
 run test_right_shift_automatic.cpp ;
 run test_right_shift_native.cpp ;
 run test_safe_compare.cpp ;
