diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index eff105b..534a2cf 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -6,8 +6,6 @@ exe example
     :
         main.cpp
     :
-        <include>../../..
-        <include>$(BOOST_ROOT)
-        <library>/boost/serialization
+        <library>/boost/foreach//boost_foreach
         <cxxflags>"-Wno-deprecated-declarations"
     ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 4770113..a894cb2 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -19,8 +19,7 @@ project
     : default-build
       <link>static
     : requirements
-      <library>/boost/test//boost_unit_test_framework
-      <include>../../..
+      <library>/boost/test//boost_unit_test_framework/<link>static
       <toolset>msvc:<define>_SCL_SECURE_NO_DEPRECATE
       # MSVC's iterator debugging causes some tests to run forever.
       <toolset>msvc:<iterator_debugging>off
@@ -28,7 +27,6 @@ project
       <toolset>gcc:<cxxflags>"-Wno-deprecated-declarations"
       <toolset>clang:<cxxflags>"-Wno-deprecated-declarations"
       <toolset>intel-linux:<cxxflags>"-Wno-deprecated-declarations"
-      <library>/boost/serialization
     ;
 
 alias accumulators_regular_tests
@@ -36,11 +34,11 @@ alias accumulators_regular_tests
       [ run covariance.cpp ]
       [ run droppable.cpp ]
       [ run error_of.cpp ]
-      [ run extended_p_square.cpp : : : <toolset>msvc-14.2,<variant>release,<cxxstd>14:<build>no ]
-      [ run extended_p_square_quantile.cpp ]
+      [ run extended_p_square.cpp : : : <toolset>msvc-14.2,<variant>release,<cxxstd>14:<build>no <library>/boost/random//boost_random ]
+      [ run extended_p_square_quantile.cpp : : : <library>/boost/random//boost_random ]
       [ run external_accumulator.cpp ]
       [ run external_weights.cpp ]
-      [ run kurtosis.cpp ]
+      [ run kurtosis.cpp : : : <library>/boost/random//boost_random ]
       [ run max.cpp ]
       [ run mean.cpp ]
       [ run median.cpp ]
@@ -57,7 +55,7 @@ alias accumulators_regular_tests
       [ run skewness.cpp ]
       [ run sum.cpp ]
       [ run sum_kahan.cpp ]
-      [ run tail.cpp ]
+      [ run tail.cpp : : : <library>/boost/foreach//boost_foreach ]
       [ run tail_mean.cpp ]
       [ run tail_quantile.cpp ]
       [ run variance.cpp ]
