diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index d0fc78b..26c6d45 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -9,7 +9,7 @@
 import os ;
 import testing ;
 
-project boost/parameter
+project
     :
         default-build
         <warnings>off
@@ -44,6 +44,7 @@ alias parameter_standard_tests
         :
         :
         :
+            <source>/boost/timer//boost_timer
         :
         :
             <variant>release
