diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index bce5f23..8d4d6ae 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,8 +5,10 @@
 # Distributed under the Boost Software License, Version 1.0.
 # (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
+import-search /boost/config/checks ;
+
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project :
     requirements
