diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index bce5f23..f000c9b 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -6,10 +6,12 @@
 # (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project :
     requirements
+        <source>/boost/config//boost_config
+
         [ requires cxx11_constexpr ]
         <toolset>clang:<cxxflags>"-pedantic -Wall -Wextra"
         <toolset>darwin:<cxxflags>"-pedantic -Wall -Wextra"
