diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index ab9cd5d..54848c5 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -7,18 +7,15 @@ project
         <toolset>msvc-8.0:<define>_SCL_SECURE_NO_DEPRECATE
         <toolset>msvc-9.0:<define>_SCL_SECURE_NO_DEPRECATE
         <toolset>msvc,<stdlib>stlport:<define>_STLP_EXPOSE_GLOBALS_IMPLEMENTATION
+        <source>/boost/assign//boost_assign
     ;
 
 exe examples 
     :
         main.cpp
-    :
-        <include>$(BOOST_ROOT)
     ;
 
 exe numbers
     :
         numbers.cpp
-    :
-        <include>$(BOOST_ROOT)
     ;
diff --git a/perf/Jamfile.v2 b/perf/Jamfile.v2
index 7769420..2038af3 100644
--- a/perf/Jamfile.v2
+++ b/perf/Jamfile.v2
@@ -8,25 +8,6 @@
 #    : requirements <library>../../../../boost/libs/regex/build//boost_regex
 #    ;
 
-BOOST_REGEX_SOURCES =
-   c_regex_traits
-   cpp_regex_traits
-   cregex
-   fileiter
-   icu
-   instances
-   posix_api
-   regex
-   regex_debug
-   regex_raw_buffer
-   regex_traits_defaults
-   static_mutex
-   w32_regex_traits
-   wc_regex_traits
-   wide_posix_api
-   winstances
-   usinstances ;
-
 exe xprperf 
     : 
         command_line.cpp
@@ -34,11 +15,10 @@ exe xprperf
         time_boost.cpp
         time_dynamic_xpressive.cpp
         time_static_xpressive.cpp
-        $(BOOST_ROOT)/libs/regex/src/$(BOOST_REGEX_SOURCES).cpp
+        /boost/regex//boost_regex
+        /boost/timer//boost_timer
+        /boost/test//boost_unit_test_framework
     :
-        <include>$(BOOST_ROOT)
-        <define>BOOST_REGEX_NO_LIB=1
-        <define>BOOST_REGEX_RECURSIVE
-        <define>BOOST_REGEX_USE_CPP_LOCALE
         <define>BOOST_XPRESSIVE_USE_CPP_TRAITS
+        <define>BOOST_TIMER_ENABLE_DEPRECATED
     ;
diff --git a/tools/Jamfile.v2 b/tools/Jamfile.v2
index ec2bb96..e715e1b 100644
--- a/tools/Jamfile.v2
+++ b/tools/Jamfile.v2
@@ -16,6 +16,4 @@ project
 exe perl2xpr
     :
         perl2xpr.cpp
-    :
-        <include>$(BOOST_ROOT)
     ;
