diff --git a/test/Jamfile b/test/Jamfile
index ffc9160..51f2371 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -8,7 +8,10 @@
 
 import testing ;
 
-project : requirements <warnings>pedantic <warnings-as-errors>on ;
+project : requirements <warnings>pedantic <warnings-as-errors>on
+    <use>/boost/core//libs
+    <use>/boost/type_traits//libs
+    ;
 
 run ios_state_unit_test.cpp ;
 run ios_state_test.cpp ;
