diff --git a/test/Jamfile b/test/Jamfile
index 873392d..7f39961 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -5,11 +5,16 @@
 import testing ;
 
 project : requirements
+  <source>/boost/container_hash//boost_container_hash
+  <source>/boost/core//boost_core
+  <source>/boost/variant2//boost_variant2
+
   <warnings>extra
   <toolset>msvc:<warnings-as-errors>on
   <toolset>clang:<warnings-as-errors>on
   <toolset>gcc:<warnings-as-errors>on
-  <toolset>clang:<cxxflags>"-Wno-unused-private-field" ;
+  <toolset>clang:<cxxflags>"-Wno-unused-private-field"
+  ;
 
 run quick.cpp ;
 
@@ -85,8 +90,8 @@ explicit describe_cxx14 ;
 
 local CXX14 = [ check-target-builds describe_cxx14 describe_cxx14 : : <build>no ] "<toolset>msvc-14.0:<cxxflags>-wd4100" ;
 
-local JSON = <library>/boost//json/<warnings>off "<toolset>msvc-14.2:<cxxflags>-wd5104" ;
-local SERIALIZATION = <library>/boost//serialization/<warnings>off "-<toolset>clang:<warnings-as-errors>on" ;
+local JSON = <library>/boost/json//boost_json/<warnings>off "<toolset>msvc-14.2:<cxxflags>-wd5104" ;
+local SERIALIZATION = <library>/boost/serialization//boost_serialization/<warnings>off "-<toolset>clang:<warnings-as-errors>on" ;
 
 run ../example/printing_enums_ct.cpp : : : $(CXX14) ;
 run ../example/printing_enums_rt.cpp : : : $(CXX14) ;
