diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 1d581a8..9b8fae3 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,9 +7,13 @@
 #
 #  See http://www.boost.org/libs/sort for library home page.
 
-import ../../config/checks/config : requires ;
+import config : requires ;
 import testing ;
 
+project : requirements
+    <source>/boost/test//boost_test_exec_monitor
+    ;
+
 {
   test-suite "sort"
     : [ run integer_sort_test.cpp
