diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index df48946..0000000
--- a/Jamfile
+++ /dev/null
@@ -1,10 +0,0 @@
-#  Boost.Logic Library Jamfile
-#
-#  Copyright (c) 2018 James E. King III
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project test ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 994f2b6..cd0cdf9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,13 +13,13 @@ import os ;
 import regex ;
 import testing ;
 
-local self = logic ;
+path-constant self : . ;
 
 rule test-expected-failures
 {
     local all_rules = ;
     local file ;
-    local tests_path = [ path.make $(BOOST_ROOT)/libs/$(self)/test/compile-fail ] ;
+    local tests_path = [ path.make $(self)/../test/compile-fail ] ;
     for file in [ path.glob-tree $(tests_path) : *.cpp ]
     {
         local rel_file = [ path.relative-to $(tests_path) $(file) ] ;
@@ -37,7 +37,7 @@ rule test-header-isolation
 {
     local all_rules = ;
     local file ;
-    local headers_path = [ path.make $(BOOST_ROOT)/libs/$(self)/include ] ;
+    local headers_path = [ path.make $(self)/../include ] ;
     for file in [ path.glob-tree $(headers_path) : *.hpp ]
     {
         local rel_file = [ path.relative-to $(headers_path) $(file) ] ;
