diff --git a/Jamfile b/Jamfile
deleted file mode 100644
index 23973f5..0000000
--- a/Jamfile
+++ /dev/null
@@ -1,10 +0,0 @@
-#  Boost.PtrContainer Library Jamfile
-#
-#  Copyright (c) 2018 James E. King III
-#
-#  Use, modification, and distribution are subject to the
-#  Boost Software License, Version 1.0. (See accompanying file
-#  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
-
-# please order by name to ease maintenance
-build-project test ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 9a3e890..f098a30 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -14,7 +14,11 @@ import testing ;
 rule sc-test ( name : extra-sources * )
 {
     return [ 
-      run $(name).cpp /boost/test//boost_unit_test_framework/<link>static $(extra-sources) ]
+      run $(name).cpp
+        /boost/test//boost_unit_test_framework/<link>static
+        /boost/lexical_cast//boost_lexical_cast
+        /boost/assign//boost_assign
+        $(extra-sources) ]
     ;
 }
 
