diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 05a0694..2473e0d 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -7,23 +7,14 @@
 #          http://www.boost.org/LICENSE_1_0.txt)
 
 import common ;
-import feature ;
 import indirect ;
 import modules ;
 import os ;
 import toolset ;
- import ../../config/checks/config : requires ;
-
-feature.feature segmented-stacks : on : optional propagated composite ;
-feature.compose <segmented-stacks>on : <define>BOOST_USE_SEGMENTED_STACKS ;
-
-feature.feature htm : tsx : optional propagated composite ;
-feature.compose <htm>tsx : <define>BOOST_USE_TSX ;
-
-feature.feature valgrind : on : optional propagated composite ;
-feature.compose <valgrind>on : <define>BOOST_USE_VALGRIND ;
+import config : requires ;
+import feature ;
 
-project boost/context
+project
     : requirements
       <target-os>windows:<define>_WIN32_WINNT=0x0601
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-fsplit-stack
@@ -54,52 +45,6 @@ project boost/context
     ;
 
 
-local rule default_binary_format ( )
-{
-    local tmp = elf ;
-    if [ os.name ] = "NT" { tmp = pe ; }
-    else if [ os.name ] = "CYGWIN" { tmp = pe ; }
-    else if [ os.name ] = "AIX" { tmp = xcoff ; }
-    else if [ os.name ] = "MACOSX" { tmp = mach-o ; }
-    return $(tmp) ;
-}
-
-feature.feature binary-format
-   : elf
-     mach-o
-     pe
-     xcoff
-   : propagated
-   ;
-feature.set-default binary-format : [ default_binary_format ] ;
-
-
-local rule default_abi ( )
-{
-    local tmp = sysv ;
-    if [ os.name ] = "NT" { tmp = ms ; }
-    else if [ os.name ] = "CYGWIN" { tmp = ms ; }
-    else if [ os.platform ] = "ARM" { tmp = aapcs ; }
-    else if [ os.platform ] = "MIPS32" { tmp = o32 ; }
-    else if [ os.platform ] = "MIPS64" { tmp = n64 ; }
-    return $(tmp) ;
-}
-
-feature.feature abi
-   : aapcs
-     eabi
-     ms
-     n32
-     n64
-     o32
-     o64
-     sysv
-     x32
-   : propagated
-   ;
-feature.set-default abi : [ default_abi ] ;
-
-
 feature.feature context-impl
     : fcontext
       ucontext
@@ -111,6 +56,7 @@ feature.set-default context-impl : fcontext ;
 feature.compose <context-impl>ucontext : <define>BOOST_USE_UCONTEXT ;
 feature.compose <context-impl>winfib : <define>BOOST_USE_WINFIB ;
 
+
 # ARM
 # ARM/AAPCS/ELF
 alias asm_sources
@@ -924,5 +870,3 @@ lib boost_context
    : impl_sources
      stack_traits_sources
    ;
-
-boost-install boost_context ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 54019a2..6be2c47 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -12,12 +12,15 @@ import modules ;
 import os ;
 import testing ;
 import toolset ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 using flags ;
 
 project boost/context/test
     : requirements
+      <library>/boost/array//boost_array
+      <library>/boost/lexical_cast//boost_lexical_cast
       <library>/boost/context//boost_context
+      <library>/boost/variant//boost_variant
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-fsplit-stack
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-DBOOST_USE_SEGMENTED_STACKS
       <toolset>clang,<segmented-stacks>on:<cxxflags>-fsplit-stack
