diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index d728ec8..a6c788f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -7,10 +7,11 @@
 #          http://www.boost.org/LICENSE_1_0.txt)
 
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project :
     requirements
+        <source>/boost/core//boost_core
         [ requires cxx14_constexpr ]
 ;
 
