diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 198de5b..168925c 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -5,11 +5,10 @@
 #    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #==============================================================================
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project hof
     : requirements [ requires cxx11_variadic_templates cxx11_constexpr ]
-      <include>../include/ 
     ;
 
 rule test_all
