diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 198de5b..33207f4 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -4,12 +4,14 @@
 #    Distributed under the Boost Software License, Version 1.0. (See accompanying
 #    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #==============================================================================
+
+import-search /boost/config/checks ;
+
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project hof
     : requirements [ requires cxx11_variadic_templates cxx11_constexpr ]
-      <include>../include/ 
     ;
 
 rule test_all
