diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 40cfe1d..83418ec 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -9,11 +9,15 @@
 import testing ;
 
 project : requirements
+  <source>/boost/core//boost_core
+  <use>/boost/exception//boost_exception
+
   <warnings>extra
   <toolset>msvc:<warnings-as-errors>on
   <toolset>clang:<warnings-as-errors>on
   <toolset>gcc:<warnings-as-errors>on
-  <toolset>gcc:<cxxflags>-Wshadow ;
+  <toolset>gcc:<cxxflags>-Wshadow
+  ;
 
 run throw_exception_test.cpp ;
 run throw_exception_no_exceptions_test.cpp ;
