diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 2ad5754d..285d2222 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -6,14 +6,16 @@
 #
 #  See http://www.boost.org/libs/serialization for the library home page.
 
-project boost/serialization
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+import config : requires ;
+
+project
     : source-location ../src
     : requirements
       <conditional>@include-spirit
 ;
 
-import ../../config/checks/config : requires ;
-
 SPIRIT_ROOT = [ modules.peek : SPIRIT_ROOT ] ;
 rule include-spirit ( properties * )
 {
@@ -144,5 +146,3 @@ lib boost_wserialization
     # switch - don't change it to BOOST_WSERIALIZATION_DYN_LINK
     <link>shared:<define>BOOST_SERIALIZATION_DYN_LINK=1
     ;
-
-boost-install boost_serialization boost_wserialization ;
diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index cce6ee65..a467dc93 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -19,7 +19,6 @@ import ../util/test :
     test-bsl-run
     test-bsl-run_archive
     test-bsl-run_files
-    test-bsl-run_polymorphic_archive
 ;
 
 test-suite "demo-suite" :
diff --git a/performance/Jamfile.v2 b/performance/Jamfile.v2
index 5ec2a296..a5295b38 100644
--- a/performance/Jamfile.v2
+++ b/performance/Jamfile.v2
@@ -21,7 +21,6 @@ import ../util/test :
     test-bsl-run
     test-bsl-run_archive
     test-bsl-run_files
-    test-bsl-run_polymorphic_archive
 ;
 
 BOOST_ARCHIVE_LIST = [ modules.peek : BOOST_ARCHIVE_LIST ] ;
@@ -33,7 +32,6 @@ test-suite "performance" :
 #    [ test-bsl-run_files performance_vector ]
 #    [ test-bsl-run_files performance_no_rtti ]
 #    [ test-bsl-run_files performance_simple_class ]
-#    [ test-bsl-run_polymorphic_archive performance_polymorphic : ../test/test_polymorphic_A ]
     
     [ test-bsl-run-no-lib performance_iterators ]
     [ test-bsl-run-no-lib performance_iterators_base64 ]
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 3e69b981..2883df3f 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -6,13 +6,14 @@
 #  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
 
-project libs/serialization/test
+project
     : id serialization_test
-    : requirements <library>/boost/filesystem
+    : requirements <library>/boost/filesystem//boost_filesystem
+        <library>/boost/math//boost_math
     ;
 
 # import rules for testing conditional on config file variables
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 # import rules from the boost serialization test
 import ../util/test :
diff --git a/util/test.jam b/util/test.jam
index 217e41b7..db11ab0c 100644
--- a/util/test.jam
+++ b/util/test.jam
@@ -11,7 +11,7 @@
 # tests.
 
 # import rules for testing conditional on config file variables
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 BOOST_ARCHIVE_LIST = [ modules.peek : BOOST_ARCHIVE_LIST ] ;
 
