diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 96e440b..3a19de1 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -59,10 +59,10 @@ test-suite iterator
     [ compile-fail minimum_category_compile_fail.cpp ]
 
     [ run next_prior_test.cpp ]
-    [ run advance_test.cpp ]
-    [ run distance_test.cpp ]
-    [ compile adl_test.cpp ]
-    [ compile range_distance_compat_test.cpp ]
+    [ run advance_test.cpp : : : <source>/boost/container//boost_container ]
+    [ run distance_test.cpp : : : <source>/boost/container//boost_container ]
+    [ compile adl_test.cpp : <source>/boost/array//boost_array ]
+    [ compile range_distance_compat_test.cpp : <source>/boost/range//boost_range ]
 
     [ run shared_iterator_test.cpp ]
 ;
