diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index dcef696..903d9d5 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -7,7 +7,7 @@
 
 # See library home page at https://www.boost.org/libs/system
 
-project boost/system
+project
     : source-location ../src
     : usage-requirements  # pass these requirement to dependents (i.e. users)
       <link>shared:<define>BOOST_SYSTEM_DYN_LINK=1
@@ -21,5 +21,3 @@ lib boost_system
    : <link>shared:<define>BOOST_SYSTEM_DYN_LINK=1
      <link>static:<define>BOOST_SYSTEM_STATIC_LINK=1
    ;
-
-boost-install boost_system ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index df74b5e..00dcf4e 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -17,6 +17,8 @@ project
 
   : requirements
 
+    <source>/boost/core//boost_core
+
     <toolset>msvc:<warnings-as-errors>on
     <toolset>gcc:<warnings-as-errors>on
     <toolset>clang:<warnings-as-errors>on
