diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index 195cbc1..bdb08ff 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -7,7 +7,7 @@
 
 # See library home page at http://www.boost.org/libs/timer
 
-project boost/timer
+project
     : source-location ../src
     : requirements
       <library>/boost/chrono//boost_chrono
@@ -24,4 +24,4 @@ lib boost_timer
      <link>static:<define>BOOST_TIMER_STATIC_LINK=1
    ;
 
-boost-install boost_timer ;
+# boost-install boost_timer ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 07cb6b6..4e41159 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -19,10 +19,10 @@ project
  
 run ../example/auto_cpu_timer_example.cpp : : : <test-info>always_show_run_output ;
 
-run cpu_timer_info.cpp : : : <test-info>always_show_run_output ;
+run cpu_timer_info.cpp : : : <test-info>always_show_run_output <source>/boost/detail//boost_detail ;
 
 run cpu_timer_test.cpp
-  : : : <test-info>always_show_run_output ;
+  : : : <test-info>always_show_run_output <source>/boost/detail//boost_detail ;
 
 run ../example/timex.cpp : echo "Hello, world" : : <test-info>always_show_run_output ;
 
@@ -32,5 +32,5 @@ run chrono_conflict_test.cpp /boost/chrono//boost_chrono : : : <link>static ;
 
 run progress_display_test.cpp ;
 
-run /boost/tools/inspect//inspect/<variant>release : $(parent) -text -brief : : <test-info>always_show_run_output : inspect ;
-explicit inspect ;
+# run /boost/tools/inspect//inspect/<variant>release : $(parent) -text -brief : : <test-info>always_show_run_output : inspect ;
+# explicit inspect ;
