diff --git a/test/Jamfile b/test/Jamfile
index 9481d5a..9921641 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -4,11 +4,14 @@
 # (See accompanying file LICENSE_1_0.txt or copy at
 # https://www.boost.org/LICENSE_1_0.txt)
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+
 import testing ;
 import path ;
 import regex ;
 import os ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : requirements
@@ -43,6 +46,8 @@ project
         <target-os>windows:<define>_CRT_SECURE_NO_DEPRECATE
     ;
 
+path-constant SCOPE_INCLUDE_DIR : ../include ;
+
 # this rule enumerates through all the sources and invokes
 # the run rule for each source, the result is a list of all
 # the run rules, which we can pass on to the test_suite rule:
@@ -53,7 +58,7 @@ rule test_all
 
     if ! [ os.environ BOOST_SCOPE_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS ]
     {
-        local headers_path = [ path.make $(BOOST_ROOT)/libs/scope/include/boost/scope ] ;
+        local headers_path = [ path.make $(SCOPE_INCLUDE_DIR)/boost/scope ] ;
         for file in [ path.glob-tree $(headers_path) : *.hpp : detail ]
         {
             local rel_file = [ path.relative-to $(headers_path) $(file) ] ;
