diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index a52dd3f..1591a9b 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -10,7 +10,7 @@ import modules ;
 import testing ;
 
 project : requirements
-
+  <source>/boost/type_traits//boost_type_traits
   <warnings>extra
   <toolset>msvc:<warnings-as-errors>on
   <toolset>clang:<warnings-as-errors>on
@@ -185,7 +185,7 @@ compile-fail scoped_enum_compile_fail_conv_to_int.cpp
 
 run underlying_type.cpp ;
 
-run fclose_deleter_test.cpp : : : <target-os>windows:<define>_CRT_SECURE_NO_WARNINGS <target-os>windows:<define>_CRT_SECURE_NO_DEPRECATE ;
+run fclose_deleter_test.cpp /boost/move//boost_move : : : <source>/boost/smart_ptr//boost_smart_ptr <target-os>windows:<define>_CRT_SECURE_NO_WARNINGS <target-os>windows:<define>_CRT_SECURE_NO_DEPRECATE ;
 
 run pointer_traits_pointer_test.cpp ;
 run pointer_traits_element_type_test.cpp ;
@@ -329,7 +329,7 @@ run sv_eq_test.cpp ;
 run sv_lt_test.cpp ;
 run sv_stream_insert_test.cpp ;
 run sv_conversion_test.cpp ;
-run sv_conversion_test2.cpp : ;
+run sv_conversion_test2.cpp /boost/utility//boost_utility : ;
 run sv_common_reference_test.cpp ;
 compile sv_common_reference_test2.cpp ;
 compile sv_windows_h_test.cpp ;
@@ -363,10 +363,10 @@ run memory_resource_test.cpp ;
 run data_test.cpp ;
 run size_test.cpp ;
 
-run serialization_nvp_test.cpp : : : <library>/boost//serialization/<warnings>off <undefined-sanitizer>norecover:<build>no ;
-run serialization_split_free_test.cpp : : : <library>/boost//serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
-run serialization_split_member_test.cpp : : : <library>/boost//serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
-run serialization_construct_data_test.cpp : : : <library>/boost//serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
+run serialization_nvp_test.cpp : : : <library>/boost/serialization//boost_serialization/<warnings>off <undefined-sanitizer>norecover:<build>no ;
+run serialization_split_free_test.cpp : : : <library>/boost/serialization//boost_serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
+run serialization_split_member_test.cpp : : : <library>/boost/serialization//boost_serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
+run serialization_construct_data_test.cpp : : : <library>/boost/serialization//boost_serialization/<warnings>off <undefined-sanitizer>norecover:<link>static ;
 
 run identity_test.cpp ;
 run identity_rvalue_test.cpp ;
