diff --git a/test/config/Jamfile.v2 b/test/config/Jamfile.v2
index 240d039..810ae1f 100644
--- a/test/config/Jamfile.v2
+++ b/test/config/Jamfile.v2
@@ -7,11 +7,13 @@
 
 import python ;
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : source-location .
     : requirements
+        <source>/boost/config//libs
+
         <define>BOOST_PFR_DETAIL_STRICT_RVALUE_TESTING=1
     ;
 
diff --git a/test/core/Jamfile.v2 b/test/core/Jamfile.v2
index ad25a84..ef360d8 100644
--- a/test/core/Jamfile.v2
+++ b/test/core/Jamfile.v2
@@ -7,11 +7,16 @@
 
 import python ;
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : source-location .
     : requirements
+        <source>/boost/config//libs
+        <source>/boost/core//libs
+        <source>/boost/functional//libs
+        <source>/boost/type_index//libs
+
         <define>BOOST_PFR_DETAIL_STRICT_RVALUE_TESTING=1
         [ requires cxx14_constexpr ]
     ;
