diff --git a/test/config/Jamfile.v2 b/test/config/Jamfile.v2
index 240d039..038145f 100644
--- a/test/config/Jamfile.v2
+++ b/test/config/Jamfile.v2
@@ -7,11 +7,13 @@
 
 import python ;
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : source-location .
     : requirements
+        <source>/boost/config//boost_config
+
         <define>BOOST_PFR_DETAIL_STRICT_RVALUE_TESTING=1
     ;
 
diff --git a/test/core/Jamfile.v2 b/test/core/Jamfile.v2
index ad25a84..31e642f 100644
--- a/test/core/Jamfile.v2
+++ b/test/core/Jamfile.v2
@@ -7,11 +7,16 @@
 
 import python ;
 import testing ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 project
     : source-location .
     : requirements
+        <source>/boost/config//boost_config
+        <source>/boost/core//boost_core
+        <source>/boost/functional//boost_functional
+        <source>/boost/type_index//boost_type_index
+
         <define>BOOST_PFR_DETAIL_STRICT_RVALUE_TESTING=1
         [ requires cxx14_constexpr ]
     ;
diff --git a/test/core/run/issue33.cpp b/test/core/run/issue33.cpp
index 009d82d..c123b2f 100644
--- a/test/core/run/issue33.cpp
+++ b/test/core/run/issue33.cpp
@@ -6,6 +6,7 @@
 // Test case for https://github.com/boostorg/pfr/issues/33
 
 #include <iostream>
+#include <memory>
 #include <vector>
 #include <boost/pfr.hpp>
 #include <boost/core/lightweight_test.hpp>
