diff --git a/build/Jamfile b/build/Jamfile
index 64695b9..156eb6c 100644
--- a/build/Jamfile
+++ b/build/Jamfile
@@ -6,7 +6,7 @@
 
 import os ;
 import feature ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 
 project : requirements
@@ -53,7 +53,7 @@ lib boost_cobalt
      [ requires 
         cxx20_hdr_concepts
      ]
-     <boost.cobalt.pmr>boost-container:<library>/boost//container
+     <boost.cobalt.pmr>boost-container:<library>/boost/container//boost_container
       [ check-target-builds
           $(config-binding:D)//cpp_lib_memory_resource
           cpp_lib_memory_resource
@@ -62,7 +62,7 @@ lib boost_cobalt
       ]
 
    : usage-requirements
-     <boost.cobalt.pmr>boost-container:<library>/boost//container
+     <boost.cobalt.pmr>boost-container:<library>/boost/container//boost_container
      <link>shared:<define>BOOST_COBALT_DYN_LINK=1
       [ check-target-builds
           $(config-binding:D)//cpp_lib_memory_resource
@@ -97,6 +97,3 @@ rule set-pmr-std ( props * )
         return <boost.cobalt.pmr>std ;
     }
 }
-
-boost-install boost_cobalt ;
-
diff --git a/example/Jamfile b/example/Jamfile
index dbfa976..ec019eb 100644
--- a/example/Jamfile
+++ b/example/Jamfile
@@ -18,12 +18,11 @@ project : requirements
   <toolset>clang-14:<boost.cobalt.pmr>boost-container
 ;
 
-exe channel      : channel.cpp         /boost//cobalt ;
-exe delay        : delay.cpp         /boost//cobalt ;
-exe delay_op     : delay_op.cpp      /boost//cobalt ;
-exe echo_server  : echo_server.cpp   /boost//cobalt ;
-exe outcome      : outcome.cpp       /boost//cobalt ;
-exe thread       : thread.cpp        /boost//cobalt ;
-exe thread_pool  : thread_pool.cpp   /boost//cobalt ;
-# exe ticker     : ticker.cpp        /boost//json      /boost//cobalt ;
-
+exe channel      : channel.cpp       /boost/cobalt//boost_cobalt ;
+exe delay        : delay.cpp         /boost/cobalt//boost_cobalt ;
+exe delay_op     : delay_op.cpp      /boost/cobalt//boost_cobalt ;
+exe echo_server  : echo_server.cpp   /boost/cobalt//boost_cobalt ;
+exe outcome      : outcome.cpp       /boost/cobalt//boost_cobalt /boost/outcome//boost_outcome ;
+exe thread       : thread.cpp        /boost/cobalt//boost_cobalt ;
+exe thread_pool  : thread_pool.cpp   /boost/cobalt//boost_cobalt ;
+# exe ticker     : ticker.cpp        /boost/cobalt//boost_cobalt /boost/json//boost_json ;
diff --git a/test/Jamfile.jam b/test/Jamfile.jam
index 450c8ea..8174849 100644
--- a/test/Jamfile.jam
+++ b/test/Jamfile.jam
@@ -20,12 +20,12 @@ project : requirements
 
 import testing ;
 
-lib test_impl : test_main.cpp /boost//cobalt /boost//unit_test_framework :
+lib test_impl : test_main.cpp /boost/cobalt//boost_cobalt /boost/test//boost_unit_test_framework :
         <link>static
         ;
 
-run main.cpp         /boost//cobalt  ;
-run main_compile.cpp /boost//cobalt util.cpp concepts.cpp ;
+run main.cpp         /boost/cobalt//boost_cobalt  ;
+run main_compile.cpp /boost/cobalt//boost_cobalt util.cpp concepts.cpp ;
 
 for local src in [ glob *.cpp : main.cpp main_compile.cpp test_main.cpp concepts.cpp util.cpp ]
 {
