diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 4241629..0c241c8 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -15,8 +15,11 @@ test-suite "examples"
     [ run     simple_bimap.cpp                                        ]
     [ run     tagged_simple_bimap.cpp                                 ]
     [ run     step_by_step.cpp                                        ]
-    [ run     population_bimap.cpp                                    ]
-    [ run     repetitions_counter.cpp                                 ]
+    [ run     population_bimap.cpp
+                    /boost/assign//boost_assign
+                    /boost/foreach//boost_foreach                     ]
+    [ run     repetitions_counter.cpp
+                    /boost/tokenizer//boost_tokenizer                 ]
     [ compile user_defined_names.cpp                                  ]
     [ run     standard_map_comparison.cpp                             ]
     [ run     at_function_examples.cpp                                ]
@@ -30,11 +33,15 @@ test-suite "examples"
 test-suite "bimap_and_boost"
     :
     [ run     bimap_and_boost/property_map.cpp                        ]
-    [ run     bimap_and_boost/range.cpp                               ]
-    [ run     bimap_and_boost/foreach.cpp                             ]
+    [ run     bimap_and_boost/range.cpp
+                    /boost/range//boost_range                         ]
+    [ run     bimap_and_boost/foreach.cpp
+                    /boost/foreach//boost_foreach                     ]
     [ run     bimap_and_boost/lambda.cpp                              ]
-    [ run     bimap_and_boost/assign.cpp                              ]
-    [ run     bimap_and_boost/xpressive.cpp                           ]
+    [ run     bimap_and_boost/assign.cpp
+                    /boost/assign//boost_assign                       ]
+    [ run     bimap_and_boost/xpressive.cpp
+                    /boost/xpressive//boost_xpressive                 ]
     [ run     bimap_and_boost/typeof.cpp                              ]
     [ run     bimap_and_boost/serialization.cpp
                     /boost/serialization//boost_serialization         ]
@@ -42,9 +49,13 @@ test-suite "bimap_and_boost"
 
 test-suite "mi_to_b_path"
     :
-    [ compile mi_to_b_path/bidirectional_map.cpp                      ]
-    [ run     mi_to_b_path/hashed_indices.cpp                         ]
+    [ compile mi_to_b_path/bidirectional_map.cpp
+                    /boost/tokenizer//boost_tokenizer                 ]
+    [ run     mi_to_b_path/hashed_indices.cpp
+                    /boost/tokenizer//boost_tokenizer                 ]
     [ compile mi_to_b_path/tagged_bidirectional_map.cpp               ]
-    [ compile mi_to_b_path/mi_bidirectional_map.cpp                   ]
-    [ run     mi_to_b_path/mi_hashed_indices.cpp                      ]
+    [ compile mi_to_b_path/mi_bidirectional_map.cpp
+                    /boost/tokenizer//boost_tokenizer                 ]
+    [ run     mi_to_b_path/mi_hashed_indices.cpp
+                    /boost/tokenizer//boost_tokenizer                 ]
     ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index f7991ef..c8b23ff 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -9,6 +9,9 @@
 # bring in rules for testing
 import testing ;
 
+project : requirements
+    <include>.
+    ;
 
 test-suite "tagged_test"
     :
@@ -41,10 +44,12 @@ test-suite "bimap_test"
     [ run test_bimap_sequenced.cpp : : :
           <toolset>gcc-4.4.7,<variant>release:<build>no                       ]
     [ run test_bimap_unconstrained.cpp                                        ]
-    [ run test_bimap_assign.cpp                                               ]
+    [ run test_bimap_assign.cpp
+          /boost/assign//boost_assign                                         ]
     [ run test_bimap_property_map.cpp                                         ]
     [ run test_bimap_modify.cpp                                               ]
-    [ run test_bimap_range.cpp                                                ]
+    [ run test_bimap_range.cpp
+          /boost/range//boost_range                                           ]
     [ run test_bimap_operator_bracket.cpp                                     ]
     [ run test_bimap_lambda.cpp                                               ]
     [ run test_bimap_mutable.cpp                                              ]
@@ -70,11 +75,16 @@ test-suite "compile_fail_test"
 test-suite "bimap_and_boost"
     :
     [ run     ../example/bimap_and_boost/property_map.cpp                     ]
-    [ run     ../example/bimap_and_boost/range.cpp                            ]
-    [ run     ../example/bimap_and_boost/foreach.cpp                          ]
+    [ run     ../example/bimap_and_boost/range.cpp
+                    /boost/range//boost_range                                 ]
+    [ run     ../example/bimap_and_boost/foreach.cpp
+                    /boost/foreach//boost_foreach                             ]
     [ run     ../example/bimap_and_boost/lambda.cpp                           ]
-    [ run     ../example/bimap_and_boost/assign.cpp                           ]
-    [ run     ../example/bimap_and_boost/xpressive.cpp : : :
+    [ run     ../example/bimap_and_boost/assign.cpp
+                    /boost/assign//boost_assign                               ]
+    [ run     ../example/bimap_and_boost/xpressive.cpp
+                    /boost/xpressive//boost_xpressive
+        : : :
         <toolset>gcc-10,<cxxstd>03:<build>no
         <toolset>gcc-11,<cxxstd>03:<build>no                                  ]
     [ run     ../example/bimap_and_boost/typeof.cpp                           ]
diff --git a/test/test_bimap_ordered.cpp b/test/test_bimap_ordered.cpp
index ec0d924..f6b62d0 100644
--- a/test/test_bimap_ordered.cpp
+++ b/test/test_bimap_ordered.cpp
@@ -34,7 +34,7 @@
 // bimap container
 #include <boost/bimap/bimap.hpp>
 
-#include <libs/bimap/test/test_bimap.hpp>
+#include <test_bimap.hpp>
 
 struct  left_tag {};
 struct right_tag {};
diff --git a/test/test_bimap_sequenced.cpp b/test/test_bimap_sequenced.cpp
index 2061627..1e868ac 100644
--- a/test/test_bimap_sequenced.cpp
+++ b/test/test_bimap_sequenced.cpp
@@ -37,7 +37,7 @@
 #include <boost/bimap/bimap.hpp>
 #include <boost/bimap/support/lambda.hpp>
 
-#include <libs/bimap/test/test_bimap.hpp>
+#include <test_bimap.hpp>
 
 struct  left_tag {};
 struct right_tag {};
diff --git a/test/test_bimap_unordered.cpp b/test/test_bimap_unordered.cpp
index 486072c..aa0120e 100644
--- a/test/test_bimap_unordered.cpp
+++ b/test/test_bimap_unordered.cpp
@@ -34,7 +34,7 @@
 // bimap container
 #include <boost/bimap/bimap.hpp>
 
-#include <libs/bimap/test/test_bimap.hpp>
+#include <test_bimap.hpp>
 
 struct  left_tag {};
 struct right_tag {};
