diff --git a/example/mpl_graph/Jamfile.v2 b/example/mpl_graph/Jamfile.v2
index cace3af..ab2e687 100644
--- a/example/mpl_graph/Jamfile.v2
+++ b/example/mpl_graph/Jamfile.v2
@@ -8,7 +8,7 @@
 
 import testing ;
 
-project msm
+project
     :
     requirements
         <include>.
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 2f566b1..8423410 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -6,12 +6,15 @@
 # Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 
+require-b2 5.0.1 ;
+import-search /boost/config/checks ;
+
 # bring in rules for testing
 import testing ;
 import os ;
-import ../../config/checks/config : requires ;
+import config : requires ;
 
-project msm
+project
     :
     requirements
         <include>.
