diff --git a/build/Jamfile.v2 b/build/Jamfile.v2
index f1a0a4d..5aa4f2d 100644
--- a/build/Jamfile.v2
+++ b/build/Jamfile.v2
@@ -8,7 +8,7 @@ import feature ;
 import modules ;
 import toolset ;
 
-project boost/coroutine
+project
     : requirements
       <library>/boost/context//boost_context
       <target-os>linux,<toolset>gcc,<segmented-stacks>on:<cxxflags>-fsplit-stack
@@ -37,7 +37,7 @@ lib boost_coroutine
     : detail/coroutine_context.cpp
       exceptions.cpp
       stack_traits_sources
-    : <link>shared:<library>../../context/build//boost_context
+    : <link>shared:<library>/boost/context//boost_context
     ;
 
-boost-install boost_coroutine ;
+# boost-install boost_coroutine ;
diff --git a/example/asymmetric/Jamfile.v2 b/example/asymmetric/Jamfile.v2
index 1b29457..0b1cf47 100644
--- a/example/asymmetric/Jamfile.v2
+++ b/example/asymmetric/Jamfile.v2
@@ -14,7 +14,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/example/asymmetric
+project
     : requirements
       <library>/boost/context//boost_context
       <library>/boost/coroutine//boost_coroutine
diff --git a/example/symmetric/Jamfile.v2 b/example/symmetric/Jamfile.v2
index 016d671..87e3769 100644
--- a/example/symmetric/Jamfile.v2
+++ b/example/symmetric/Jamfile.v2
@@ -14,7 +14,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/example/symmetric
+project
     : requirements
       <library>/boost/context//boost_context
       <library>/boost/coroutine//boost_coroutine
diff --git a/performance/asymmetric/Jamfile.v2 b/performance/asymmetric/Jamfile.v2
index aa2ffb3..526be6b 100644
--- a/performance/asymmetric/Jamfile.v2
+++ b/performance/asymmetric/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/performance/asymmetric
+project
     : requirements
       <library>/boost/chrono//boost_chrono
       <library>/boost/context//boost_context
diff --git a/performance/asymmetric/segmented/Jamfile.v2 b/performance/asymmetric/segmented/Jamfile.v2
index 073f7e7..d1f8036 100644
--- a/performance/asymmetric/segmented/Jamfile.v2
+++ b/performance/asymmetric/segmented/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/performance/segmented
+project
     : requirements
       <library>/boost/chrono//boost_chrono
       <library>/boost/context//boost_context
diff --git a/performance/symmetric/Jamfile.v2 b/performance/symmetric/Jamfile.v2
index 93c239d..526be6b 100644
--- a/performance/symmetric/Jamfile.v2
+++ b/performance/symmetric/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/performance/symmetric
+project
     : requirements
       <library>/boost/chrono//boost_chrono
       <library>/boost/context//boost_context
diff --git a/performance/symmetric/segmented/Jamfile.v2 b/performance/symmetric/segmented/Jamfile.v2
index 073f7e7..d1f8036 100644
--- a/performance/symmetric/segmented/Jamfile.v2
+++ b/performance/symmetric/segmented/Jamfile.v2
@@ -13,7 +13,7 @@ import modules ;
 import os ;
 import toolset ;
 
-project boost/coroutine/performance/segmented
+project
     : requirements
       <library>/boost/chrono//boost_chrono
       <library>/boost/context//boost_context
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 4326bf2..96b0a46 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -13,7 +13,7 @@ import os ;
 import testing ;
 import toolset ;
 
-project boost/coroutine/test
+project
     : requirements
       <library>/boost/context//boost_context
       <library>/boost/coroutine//boost_coroutine
