diff --git a/test/Jamfile b/test/Jamfile
index 53c60ec..b97bee1 100644
--- a/test/Jamfile
+++ b/test/Jamfile
@@ -9,6 +9,7 @@ build-project container_fwd ;
 
 project detail/test
   : requirements
+    <library>/boost/type_traits//boost_type_traits
     <c++-template-depth>300
     <toolset>clang:<cxxflags>-Wno-unused
     <toolset>clang:<cxxflags>-Wno-tautological-compare
@@ -18,13 +19,13 @@ project detail/test
   ;
 
 # import rules for testing conditional on config file variables
-import ../../config/checks/config : requires ;
+import config : requires ;
 
 run binary_search_test.cpp ;
 run blank_test.cpp ;
 run is_sorted_test.cpp ;
 run numeric_traits_test.cpp ;
-run is_xxx_test.cpp ;
+run is_xxx_test.cpp /boost/preprocessor//boost_preprocessor ;
 # run test_utf8_codecvt.cpp : : : [ requires std_wstreambuf ] ;
 run test_utf8_codecvt.cpp ;
 run allocator_utilities_test.cpp ;
