diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index ef1260e..0fb0fd9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -15,7 +15,7 @@ compile-fail fail_csubarray.cpp ;
 compile-fail fail_csubarray2.cpp ;
 compile-fail fail_csubarray3.cpp ;
 compile-fail fail_cview.cpp ;
-compile-fail fail_cview2.cpp ;
+compile-fail fail_cview2.cpp : <source>/boost/test//boost_test ;
 compile-fail fail_cview3.cpp ;
 compile-fail fail_ref_cbracket.cpp ;
 compile-fail fail_ref_cdata.cpp ;
