diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index 8b80c06..2cc17d9 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -16,6 +16,7 @@ project preprocessor_tests : requirements <warnings>on
                                           <toolset>gcc-4.5.2:<warnings>all
                                           <toolset>gcc-4.5.2:<cxxflags>-Wno-variadic-macros
                                           <toolset>msvc:<warnings>all
+                                          <include>.
                                           ;
 
 alias preprocessor : : 
diff --git a/test/arithmetic.c b/test/arithmetic.c
index 53a071e..f9bcc93 100644
--- a/test/arithmetic.c
+++ b/test/arithmetic.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/arithmetic.cxx>
+# include "arithmetic.cxx"
diff --git a/test/arithmetic.cpp b/test/arithmetic.cpp
index 53a071e..f9bcc93 100644
--- a/test/arithmetic.cpp
+++ b/test/arithmetic.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/arithmetic.cxx>
+# include "arithmetic.cxx"
diff --git a/test/arithmetic.cxx b/test/arithmetic.cxx
index 3290dc2..caf509b 100644
--- a/test/arithmetic.cxx
+++ b/test/arithmetic.cxx
@@ -11,7 +11,7 @@
 #
 # include <boost/preprocessor/config/limits.hpp>
 # include <boost/preprocessor/arithmetic.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 /* addition */
 
diff --git a/test/array.c b/test/array.c
index 9150747..725a6f1 100644
--- a/test/array.c
+++ b/test/array.c
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/array.cxx>
+# include "array.cxx"
diff --git a/test/array.cpp b/test/array.cpp
index 9150747..725a6f1 100644
--- a/test/array.cpp
+++ b/test/array.cpp
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/array.cxx>
+# include "array.cxx"
diff --git a/test/array.cxx b/test/array.cxx
index 895b7a6..c7e3be9 100644
--- a/test/array.cxx
+++ b/test/array.cxx
@@ -13,7 +13,7 @@
 #
 # include <boost/preprocessor/config/limits.hpp>
 # include <boost/preprocessor/array.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 # include <boost/preprocessor/facilities/is_empty.hpp>
 # include <boost/preprocessor/list/at.hpp>
 # include <boost/preprocessor/list/size.hpp>
diff --git a/test/checkempty.cpp b/test/checkempty.cpp
index 6306708..62a55fd 100644
--- a/test/checkempty.cpp
+++ b/test/checkempty.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/checkempty.cxx>
+# include "checkempty.cxx"
diff --git a/test/checkempty.cxx b/test/checkempty.cxx
index ef32da1..c642111 100644
--- a/test/checkempty.cxx
+++ b/test/checkempty.cxx
@@ -9,7 +9,7 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 # include <boost/preprocessor/facilities/check_empty.hpp>
 
 # if BOOST_PP_VARIADIC_HAS_OPT()
diff --git a/test/clang_cuda.cu b/test/clang_cuda.cu
index e13d656..c360a38 100644
--- a/test/clang_cuda.cu
+++ b/test/clang_cuda.cu
@@ -9,7 +9,7 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/test_macro.h>
+# include "test_macro.h"
 
 #if defined(__clang__) && defined(__CUDACC__) && defined(__CUDA__)
 
diff --git a/test/comparison.c b/test/comparison.c
index 5f7daea..170b636 100644
--- a/test/comparison.c
+++ b/test/comparison.c
@@ -9,5 +9,5 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/comparison.cxx>
+# include "comparison.cxx"
 
diff --git a/test/comparison.cpp b/test/comparison.cpp
index 40720c6..c812b2d 100644
--- a/test/comparison.cpp
+++ b/test/comparison.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/comparison.cxx>
+# include "comparison.cxx"
diff --git a/test/comparison.cxx b/test/comparison.cxx
index 0e7ff9d..65ae94d 100644
--- a/test/comparison.cxx
+++ b/test/comparison.cxx
@@ -11,7 +11,7 @@
 #
 # include <boost/preprocessor/config/limits.hpp>
 # include <boost/preprocessor/comparison.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 /* equality */
 
diff --git a/test/control.c b/test/control.c
index db729d9..786b4dc 100644
--- a/test/control.c
+++ b/test/control.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/control.cxx>
+# include "control.cxx"
diff --git a/test/control.cpp b/test/control.cpp
index db729d9..786b4dc 100644
--- a/test/control.cpp
+++ b/test/control.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/control.cxx>
+# include "control.cxx"
diff --git a/test/control.cxx b/test/control.cxx
index e449ed1..8a2b7c8 100644
--- a/test/control.cxx
+++ b/test/control.cxx
@@ -13,7 +13,7 @@
 # include <boost/preprocessor/arithmetic/add.hpp>
 # include <boost/preprocessor/arithmetic/dec.hpp>
 # include <boost/preprocessor/control.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 # define TR(x) 1
 
diff --git a/test/debug.c b/test/debug.c
index 7325e33..8250498 100644
--- a/test/debug.c
+++ b/test/debug.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/debug.cxx>
+# include "debug.cxx"
diff --git a/test/debug.cpp b/test/debug.cpp
index 7325e33..8250498 100644
--- a/test/debug.cpp
+++ b/test/debug.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/debug.cxx>
+# include "debug.cxx"
diff --git a/test/debug.cxx b/test/debug.cxx
index 6c4c5fa..6495fc8 100644
--- a/test/debug.cxx
+++ b/test/debug.cxx
@@ -10,7 +10,7 @@
 # /* See http://www.boost.org for most recent version. */
 #
 # include <boost/preprocessor/debug.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 BEGIN sizeof(BOOST_PP_ASSERT_MSG(0, "text") "") / sizeof(char) != 1 END
 BEGIN sizeof(BOOST_PP_ASSERT_MSG(1, "text") "") / sizeof(char) == 1 END
diff --git a/test/facilities.c b/test/facilities.c
index 25cef08..5a56359 100644
--- a/test/facilities.c
+++ b/test/facilities.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/facilities.cxx>
+# include "facilities.cxx"
diff --git a/test/facilities.cpp b/test/facilities.cpp
index 25cef08..5a56359 100644
--- a/test/facilities.cpp
+++ b/test/facilities.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/facilities.cxx>
+# include "facilities.cxx"
diff --git a/test/facilities.cxx b/test/facilities.cxx
index dbc45d9..0c402fd 100644
--- a/test/facilities.cxx
+++ b/test/facilities.cxx
@@ -13,7 +13,7 @@
 # include <boost/preprocessor/facilities.hpp>
 # include <boost/preprocessor/arithmetic/add.hpp>
 # include <boost/preprocessor/arithmetic/mul.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 BEGIN BOOST_PP_APPLY(BOOST_PP_NIL) 0 == 0 END
 BEGIN BOOST_PP_APPLY((0)) == 0 END
diff --git a/test/isempty.c b/test/isempty.c
index bd4c2c6..d0a4133 100644
--- a/test/isempty.c
+++ b/test/isempty.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty.cxx>
+# include "isempty.cxx"
diff --git a/test/isempty.cpp b/test/isempty.cpp
index bd4c2c6..d0a4133 100644
--- a/test/isempty.cpp
+++ b/test/isempty.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty.cxx>
+# include "isempty.cxx"
diff --git a/test/isempty.cxx b/test/isempty.cxx
index 073d0ae..f6e3aa7 100644
--- a/test/isempty.cxx
+++ b/test/isempty.cxx
@@ -15,7 +15,7 @@
 
 # include <boost/preprocessor/facilities/empty.hpp>
 # include <boost/preprocessor/facilities/is_empty.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #define DATA
 #define OBJECT OBJECT2
diff --git a/test/isempty_variadic_standard_failure.c b/test/isempty_variadic_standard_failure.c
index 36f6ff7..45fa207 100644
--- a/test/isempty_variadic_standard_failure.c
+++ b/test/isempty_variadic_standard_failure.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty_variadic_standard_failure.cxx>
+# include "isempty_variadic_standard_failure.cxx"
diff --git a/test/isempty_variadic_standard_failure.cpp b/test/isempty_variadic_standard_failure.cpp
index 36f6ff7..45fa207 100644
--- a/test/isempty_variadic_standard_failure.cpp
+++ b/test/isempty_variadic_standard_failure.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty_variadic_standard_failure.cxx>
+# include "isempty_variadic_standard_failure.cxx"
diff --git a/test/isempty_variadic_standard_failure.cxx b/test/isempty_variadic_standard_failure.cxx
index a6258e4..1bfe0ee 100644
--- a/test/isempty_variadic_standard_failure.cxx
+++ b/test/isempty_variadic_standard_failure.cxx
@@ -10,7 +10,7 @@
 # /* See http://www.boost.org for most recent version. */
 #
 # include <boost/preprocessor/facilities/is_empty.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #if (BOOST_PP_CONFIG_FLAGS() & BOOST_PP_CONFIG_STRICT()) && !BOOST_PP_VARIADICS_MSVC
 
diff --git a/test/isempty_variadic_standard_failure2.c b/test/isempty_variadic_standard_failure2.c
index a2aa690..6be243d 100644
--- a/test/isempty_variadic_standard_failure2.c
+++ b/test/isempty_variadic_standard_failure2.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty_variadic_standard_failure2.cxx>
+# include "isempty_variadic_standard_failure2.cxx"
diff --git a/test/isempty_variadic_standard_failure2.cpp b/test/isempty_variadic_standard_failure2.cpp
index a2aa690..6be243d 100644
--- a/test/isempty_variadic_standard_failure2.cpp
+++ b/test/isempty_variadic_standard_failure2.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/isempty_variadic_standard_failure2.cxx>
+# include "isempty_variadic_standard_failure2.cxx"
diff --git a/test/isempty_variadic_standard_failure2.cxx b/test/isempty_variadic_standard_failure2.cxx
index be92d47..ae067f5 100644
--- a/test/isempty_variadic_standard_failure2.cxx
+++ b/test/isempty_variadic_standard_failure2.cxx
@@ -10,7 +10,7 @@
 # /* See http://www.boost.org for most recent version. */
 #
 # include <boost/preprocessor/facilities/is_empty.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #if (BOOST_PP_CONFIG_FLAGS() & BOOST_PP_CONFIG_STRICT()) && !BOOST_PP_VARIADICS_MSVC
 
diff --git a/test/iteration.cpp b/test/iteration.cpp
index f06be5d..2b551eb 100644
--- a/test/iteration.cpp
+++ b/test/iteration.cpp
@@ -13,7 +13,7 @@
 #
 # if !BOOST_PP_IS_SELFISH
 #
-# include <libs/preprocessor/test/iteration.h>
+# include "iteration.h"
 #
 # define TEST(n) BEGIN n == n END
 #
@@ -37,7 +37,7 @@
 #
 # endif
 #
-# define BOOST_PP_INDIRECT_SELF <libs/preprocessor/test/iteration.cpp>
+# define BOOST_PP_INDIRECT_SELF "iteration.cpp"
 # include BOOST_PP_INCLUDE_SELF()
 #
 # else
diff --git a/test/iteration.h b/test/iteration.h
index 035531a..37bda60 100644
--- a/test/iteration.h
+++ b/test/iteration.h
@@ -16,19 +16,19 @@
 # include <boost/preprocessor/control/expr_iif.hpp>
 # include <boost/preprocessor/iteration.hpp>
 # include <boost/preprocessor/logical/bitor.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 #
 # define NO_FLAGS
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (3, (1, 10, <libs/preprocessor/test/iteration.h>))
+# define BOOST_PP_ITERATION_PARAMS_1 (3, (1, 10, "iteration.h"))
 # include BOOST_PP_ITERATE()
 #
 # undef NO_FLAGS
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (1, 5, <libs/preprocessor/test/iteration.h>, 0x0001))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (1, 5, "iteration.h", 0x0001))
 # include BOOST_PP_ITERATE()
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (1, 5, <libs/preprocessor/test/iteration.h>, 0x0002))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (1, 5, "iteration.h", 0x0002))
 # include BOOST_PP_ITERATE()
 #
 # if BOOST_PP_LIMIT_ITERATION == 512
@@ -53,7 +53,7 @@
             )                              \
 /* */
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (0, 512, <libs/preprocessor/test/iteration.h>, 0x0003))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (0, 512, "iteration.h", 0x0003))
 # include BOOST_PP_ITERATE()
 #
 # undef ITER100S
@@ -78,7 +78,7 @@
             )                              \
 /* */
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (512, 0, <libs/preprocessor/test/iteration.h>, 0x0004))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (512, 0, "iteration.h", 0x0004))
 # include BOOST_PP_ITERATE()
 #
 # undef ITER50S
@@ -127,7 +127,7 @@
              )                                 \
 /* */
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (0, 1024, <libs/preprocessor/test/iteration.h>, 0x0005))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (0, 1024, "iteration.h", 0x0005))
 # include BOOST_PP_ITERATE()
 #
 # undef ITER100SA
@@ -172,7 +172,7 @@
              )                                 \
 /* */
 #
-# define BOOST_PP_ITERATION_PARAMS_1 (4, (1024, 0, <libs/preprocessor/test/iteration.h>, 0x0006))
+# define BOOST_PP_ITERATION_PARAMS_1 (4, (1024, 0, "iteration.h", 0x0006))
 # include BOOST_PP_ITERATE()
 #
 # undef ITER50SA
@@ -197,7 +197,7 @@ struct BOOST_PP_CAT(Y, BOOST_PP_ITERATION()) { };
 # elif BOOST_PP_ITERATION_DEPTH() == 1 \
     && BOOST_PP_ITERATION_FLAGS() == 0x0002
 
-# define BOOST_PP_ITERATION_PARAMS_2 (3, (1, BOOST_PP_ITERATION(), <libs/preprocessor/test/iteration.h>))
+# define BOOST_PP_ITERATION_PARAMS_2 (3, (1, BOOST_PP_ITERATION(), "iteration.h"))
 # include BOOST_PP_ITERATE()
 
 # elif BOOST_PP_ITERATION_DEPTH() == 1 \
diff --git a/test/list.c b/test/list.c
index 23bb7b1..e97f2d1 100644
--- a/test/list.c
+++ b/test/list.c
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/list.cxx>
+# include "list.cxx"
diff --git a/test/list.cpp b/test/list.cpp
index 23bb7b1..e97f2d1 100644
--- a/test/list.cpp
+++ b/test/list.cpp
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/list.cxx>
+# include "list.cxx"
diff --git a/test/list.cxx b/test/list.cxx
index c9f8583..ce62109 100644
--- a/test/list.cxx
+++ b/test/list.cxx
@@ -26,7 +26,7 @@
 # include <boost/preprocessor/seq/elem.hpp>
 # include <boost/preprocessor/seq/size.hpp>
 # include <boost/preprocessor/variadic/elem.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 # define LISTNIL BOOST_PP_NIL
 # define LIST (4, (1, (5, (2, BOOST_PP_NIL))))
diff --git a/test/logical.c b/test/logical.c
index 9f50287..666241f 100644
--- a/test/logical.c
+++ b/test/logical.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/logical.cxx>
+# include "logical.cxx"
diff --git a/test/logical.cpp b/test/logical.cpp
index 9f50287..666241f 100644
--- a/test/logical.cpp
+++ b/test/logical.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/logical.cxx>
+# include "logical.cxx"
diff --git a/test/logical.cxx b/test/logical.cxx
index 755d2a6..08c0216 100644
--- a/test/logical.cxx
+++ b/test/logical.cxx
@@ -11,7 +11,7 @@
 #
 # include <boost/preprocessor/config/limits.hpp>
 # include <boost/preprocessor/logical.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 BEGIN BOOST_PP_NOT(0) == 1 END
 BEGIN BOOST_PP_NOT(2) == 0 END
diff --git a/test/punctuation.c b/test/punctuation.c
index 8fc5dfc..410ec7b 100644
--- a/test/punctuation.c
+++ b/test/punctuation.c
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/punctuation.cxx>
+# include "punctuation.cxx"
diff --git a/test/punctuation.cpp b/test/punctuation.cpp
index 8fc5dfc..410ec7b 100644
--- a/test/punctuation.cpp
+++ b/test/punctuation.cpp
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/punctuation.cxx>
+# include "punctuation.cxx"
diff --git a/test/punctuation.cxx b/test/punctuation.cxx
index 67bc0b8..ea198f5 100644
--- a/test/punctuation.cxx
+++ b/test/punctuation.cxx
@@ -10,7 +10,7 @@
 # /* See http://www.boost.org for most recent version. */
 #
 # include <boost/preprocessor/punctuation.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 # define A_TUPLE (*,#,zz)
 # define A_TUPLE2 (*,#,(zz,44,(e7)))
diff --git a/test/quick.cpp b/test/quick.cpp
index 73372a3..56e7c28 100644
--- a/test/quick.cpp
+++ b/test/quick.cpp
@@ -4,7 +4,7 @@
 // https://www.boost.org/LICENSE_1_0.txt
 
 #include <boost/preprocessor/comparison.hpp>
-#include "test_macro.h"
+# include "test_macro.h"
 
 /* equality */
 
diff --git a/test/repetition.cpp b/test/repetition.cpp
index f35d394..4147878 100644
--- a/test/repetition.cpp
+++ b/test/repetition.cpp
@@ -18,7 +18,7 @@
 # include <boost/preprocessor/facilities/intercept.hpp>
 # include <boost/preprocessor/logical/bitor.hpp>
 # include <boost/preprocessor/repetition.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 # define MAX 10
 
diff --git a/test/selection.c b/test/selection.c
index f8fdaba..b21282c 100644
--- a/test/selection.c
+++ b/test/selection.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/selection.cxx>
+# include "selection.cxx"
diff --git a/test/selection.cpp b/test/selection.cpp
index f8fdaba..b21282c 100644
--- a/test/selection.cpp
+++ b/test/selection.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/selection.cxx>
+# include "selection.cxx"
diff --git a/test/selection.cxx b/test/selection.cxx
index 6891b28..720a776 100644
--- a/test/selection.cxx
+++ b/test/selection.cxx
@@ -11,7 +11,7 @@
 #
 # include <boost/preprocessor/config/limits.hpp>
 # include <boost/preprocessor/selection.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 BEGIN BOOST_PP_MAX(2, 2) == 2 END
 BEGIN BOOST_PP_MAX(2, 1) == 2 END
diff --git a/test/seq.c b/test/seq.c
index e36c5d1..6200def 100644
--- a/test/seq.c
+++ b/test/seq.c
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/seq.cxx>
+# include "seq.cxx"
diff --git a/test/seq.cpp b/test/seq.cpp
index e36c5d1..6200def 100644
--- a/test/seq.cpp
+++ b/test/seq.cpp
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/seq.cxx>
+# include "seq.cxx"
diff --git a/test/seq.cxx b/test/seq.cxx
index f96e7f9..26291a7 100644
--- a/test/seq.cxx
+++ b/test/seq.cxx
@@ -36,7 +36,7 @@
 # include <boost/preprocessor/variadic/elem.hpp>
 # include <boost/preprocessor/variadic/size.hpp>
 # include <boost/preprocessor/variadic/has_opt.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 # define SEQ_NONE ()
 # define SEQ (4)(1)(5)(2)
diff --git a/test/slot.c b/test/slot.c
index 4b97bcd..3cc1b2d 100644
--- a/test/slot.c
+++ b/test/slot.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/slot.cxx>
+# include "slot.cxx"
diff --git a/test/slot.cpp b/test/slot.cpp
index 4b97bcd..3cc1b2d 100644
--- a/test/slot.cpp
+++ b/test/slot.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/slot.cxx>
+# include "slot.cxx"
diff --git a/test/slot.cxx b/test/slot.cxx
index 2b5b3f8..c26cfe1 100644
--- a/test/slot.cxx
+++ b/test/slot.cxx
@@ -10,7 +10,7 @@
 # /* See http://www.boost.org for most recent version. */
 #
 # include <boost/preprocessor/slot.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 # include <boost/preprocessor/slot/counter.hpp>
 
 # define X() 4
diff --git a/test/stringize.c b/test/stringize.c
index 0ac19ec..d00111c 100644
--- a/test/stringize.c
+++ b/test/stringize.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/stringize.cxx>
+# include "stringize.cxx"
diff --git a/test/stringize.cpp b/test/stringize.cpp
index 0ac19ec..d00111c 100644
--- a/test/stringize.cpp
+++ b/test/stringize.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/stringize.cxx>
+# include "stringize.cxx"
diff --git a/test/stringize.cxx b/test/stringize.cxx
index c21d1f6..8c8b763 100644
--- a/test/stringize.cxx
+++ b/test/stringize.cxx
@@ -15,7 +15,7 @@
 # if !defined __cplusplus
 #include <wchar.h>
 #endif
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #define VDATA 1,2,3,4
 #define NDATA
diff --git a/test/test.h b/test/test.h
index d1d60c3..1fc33af 100644
--- a/test/test.h
+++ b/test/test.h
@@ -14,7 +14,7 @@
 # ifndef BOOST_LIBS_PREPROCESSOR_REGRESSION_TEST_H
 # define BOOST_LIBS_PREPROCESSOR_REGRESSION_TEST_H
 #
-# include <libs/preprocessor/test/test_macro.h>
-# include <libs/preprocessor/test/test_main.h>
+# include "test_macro.h"
+#include "test_main.h"
 #
 # endif
diff --git a/test/tuple.c b/test/tuple.c
index e9864a0..dc4f932 100644
--- a/test/tuple.c
+++ b/test/tuple.c
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/tuple.cxx>
+# include "tuple.cxx"
diff --git a/test/tuple.cpp b/test/tuple.cpp
index e9864a0..dc4f932 100644
--- a/test/tuple.cpp
+++ b/test/tuple.cpp
@@ -11,4 +11,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/tuple.cxx>
+# include "tuple.cxx"
diff --git a/test/tuple.cxx b/test/tuple.cxx
index bfcdecb..08c12a4 100644
--- a/test/tuple.cxx
+++ b/test/tuple.cxx
@@ -25,8 +25,8 @@
 # include <boost/preprocessor/variadic/size.hpp>
 # include <boost/preprocessor/variadic/elem.hpp>
 # include <boost/preprocessor/variadic/has_opt.hpp>
-# include <libs/preprocessor/test/test.h>
-# include <libs/preprocessor/test/tuple_elem_bug_test.cxx>
+# include "test.h"
+# include "tuple_elem_bug_test.cxx"
 
 # define TUPLE (0, 1, 2, 3, 4, 5)
 # define TUPLE_NONE ()
diff --git a/test/tuple_elem_bug_test.cxx b/test/tuple_elem_bug_test.cxx
index c365609..217a6dc 100644
--- a/test/tuple_elem_bug_test.cxx
+++ b/test/tuple_elem_bug_test.cxx
@@ -12,7 +12,7 @@
 # include <boost/preprocessor/cat.hpp>
 # include <boost/preprocessor/control/if.hpp>
 # include <boost/preprocessor/tuple.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #define TN_GEN_ONE(p) (1)
 #define TN_GEN_ZERO(p) (0)
diff --git a/test/vaopt.cpp b/test/vaopt.cpp
index 9801a19..4eb6069 100644
--- a/test/vaopt.cpp
+++ b/test/vaopt.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/vaopt.cxx>
+# include "vaopt.cxx"
diff --git a/test/vaopt.cxx b/test/vaopt.cxx
index 6cb8702..a0f9659 100644
--- a/test/vaopt.cxx
+++ b/test/vaopt.cxx
@@ -9,7 +9,7 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 # include <boost/preprocessor/variadic/has_opt.hpp>
 
 # if BOOST_PP_VARIADIC_HAS_OPT()
diff --git a/test/variadic.c b/test/variadic.c
index a9d2532..385f4bb 100644
--- a/test/variadic.c
+++ b/test/variadic.c
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/variadic.cxx>
+# include "variadic.cxx"
diff --git a/test/variadic.cpp b/test/variadic.cpp
index a9d2532..385f4bb 100644
--- a/test/variadic.cpp
+++ b/test/variadic.cpp
@@ -9,4 +9,4 @@
 #
 # /* See http://www.boost.org for most recent version. */
 #
-# include <libs/preprocessor/test/variadic.cxx>
+# include "variadic.cxx"
diff --git a/test/variadic.cxx b/test/variadic.cxx
index 2646cc3..732ea78 100644
--- a/test/variadic.cxx
+++ b/test/variadic.cxx
@@ -19,7 +19,7 @@
 # include <boost/preprocessor/seq/size.hpp>
 # include <boost/preprocessor/tuple/size.hpp>
 # include <boost/preprocessor/tuple/elem.hpp>
-# include <libs/preprocessor/test/test.h>
+# include "test.h"
 
 #define VDATA 0,1,2,3,4,5,6
 #define VDATA_LARGE 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32
