diff --git a/Jamfile.v2 b/Jamfile.v2
deleted file mode 100644
index 8333597..0000000
--- a/Jamfile.v2
+++ /dev/null
@@ -1,5 +0,0 @@
-import testing ;
-
-# Tests from Jamfiles in individual library test subdirectories
-build-project example ;   # test-suite interprocess_example
-build-project test ;      # test-suite interprocess_test
\ No newline at end of file
diff --git a/example/Jamfile.v2 b/example/Jamfile.v2
index 22da3e0..223d906 100644
--- a/example/Jamfile.v2
+++ b/example/Jamfile.v2
@@ -15,38 +15,28 @@
 # the run rule for each source, the result is a list of all
 # the run rules, which we can pass on to the test_suite rule:
 
+project : requirements
+    <toolset>acc:<linkflags>-lrt
+    <toolset>acc-pa_risc:<linkflags>-lrt
+    <toolset>gcc,<target-os>windows:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+    <target-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
+    <target-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+    <target-os>linux:<linkflags>"-lrt"
+    <library>/boost/multi_index//boost_multi_index
+    ;
+
 rule test_all
 {
    local all_rules = ;
 
    for local fileb in [ glob comp*.cpp ]
    {
-      all_rules += [ link $(fileb)
-      :  # additional args
-        <toolset>acc:<linkflags>-lrt
-        <toolset>acc-pa_risc:<linkflags>-lrt
-        <toolset>gcc,<target-os>windows:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
-        <target-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>linux:<linkflags>"-lrt"
-      :  # test-files
-      :  # requirements
-      ] ;
+      all_rules += [ link $(fileb) ] ;
    }
 
    for local fileb in [ glob doc_*.cpp ]
    {
-      all_rules += [ run $(fileb)
-      :  # additional args
-      :  # test-files
-      :  # requirements
-        <toolset>acc:<linkflags>-lrt
-        <toolset>acc-pa_risc:<linkflags>-lrt
-        <toolset>gcc-mingw:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
-        <target-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>linux:<linkflags>"-lrt"
-      ] ;
+      all_rules += [ run $(fileb) ] ;
    }
 
    return $(all_rules) ;
diff --git a/test/Jamfile.v2 b/test/Jamfile.v2
index a09bf8f..b2f54cf 100644
--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -15,23 +15,25 @@
 # the run rule for each source, the result is a list of all
 # the run rules, which we can pass on to the test_suite rule:
 
+project : requirements
+    <source>/boost/date_time//boost_date_time
+    <source>/boost/multi_index//boost_multi_index
+    <toolset>acc:<linkflags>-lrt
+    <toolset>acc-pa_risc:<linkflags>-lrt
+    <toolset>gcc,<target-os>windows:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+    <target-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
+    <target-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
+    <target-os>linux:<linkflags>"-lrt"
+    <library>/boost/chrono//boost_chrono
+    ;
+
 rule test_all
 {
    local all_rules = ;
 
    for local fileb in [ glob *.cpp ]
    {
-      all_rules += [ run $(fileb)
-      :  # additional args
-      :  # test-files
-      :  # requirements
-        <toolset>acc:<linkflags>-lrt
-        <toolset>acc-pa_risc:<linkflags>-lrt
-        <toolset>gcc,<target-os>windows:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>hpux,<toolset>gcc:<linkflags>"-Wl,+as,mpas"
-        <target-os>windows,<toolset>clang:<linkflags>"-lole32 -loleaut32 -lpsapi -ladvapi32"
-        <target-os>linux:<linkflags>"-lrt"
-      ] ;
+      all_rules += [ run $(fileb) ] ;
    }
 
    return $(all_rules) ;
